# Quantifying penetrance in a dominant disease gene using large population control cohorts

<a name="top"></a>

<b>This is the author's version of the work. It is posted here by permission of the AAAS for personal use, not for redistribution. The definitive version was published as: [Minikel et al. Quantifying prion disease penetrance using large population control cohorts. Sci. Transl. Med. 8, 322ra9 (2016). DOI: 10.1126/scitranslmed.aad5169](http://stm.sciencemag.org/content/8/322/322ra9.full).</b>

Eric Vallabh Minikel†<sup>1,2,3,4</sup>, Sonia M. Vallabh<sup>1,3,4</sup>, Monkol Lek<sup>1,2</sup>, Karol Estrada<sup>1,2</sup>, Kaitlin E. Samocha<sup>1,2,3</sup>, J. Fah Sathirapongsasuti<sup>5</sup>, Cory Y. McLean<sup>5</sup>, Joyce Y. Tung<sup>5</sup>, Linda P.C. Yu<sup>5</sup>, Pierluigi Gambetti<sup>6</sup>, Janis Blevins<sup>6</sup>, Shulin Zhang<sup>7</sup>, Yvonne Cohen<sup>6</sup>, Wei Chen<sup>6</sup>, Masahito Yamada<sup>8</sup>, Tsuyoshi Hamaguchi<sup>8</sup>, Nobuo Sanjo<sup>9</sup>, Hidehiro Mizusawa<sup>10</sup>, Yosikazu Nakamura<sup>11</sup>, Tetsuyuki Kitamoto<sup>12</sup>, Steven J. Collins<sup>13</sup>, Alison Boyd<sup>13</sup>, Robert G. Will<sup>14</sup>, Richard Knight<sup>14</sup>, Claudia Ponto<sup>15</sup>, Inga Zerr<sup>15</sup>, Theo F.J. Kraus<sup>16</sup>, Sabina Eigenbrod<sup>16</sup>, Armin Giese<sup>16</sup>, Miguel Calero<sup>17</sup>, Jesús de Pedro-Cuesta<sup>17</sup>, Stéphane Haïk<sup>18,19</sup>, Jean-Louis Laplanche<sup>20</sup>, Elodie Bouaziz-Amar<sup>20</sup>, Jean-Philippe Brandel<sup>18,19</sup>, Sabina Capellari<sup>21,22</sup>, Piero Parchi<sup>21,22</sup>, Anna Poleggi<sup>23</sup>, Anna Ladogana<sup>23</sup>, Anne H. O'Donnell-Luria<sup>2,1,24</sup>, Konrad J. Karczewski<sup>2,1</sup>, Jamie L. Marshall<sup>1,2</sup>, Michael Boehnke<sup>25</sup>, Markku Laakso<sup>26</sup>, Karen L. Mohlke<sup>27</sup>, Anna Kähler<sup>28</sup>, Kimberly Chambert<sup>29</sup>, Steven McCarroll<sup>29</sup>, Patrick F. Sullivan<sup>27,28</sup>, Christina M. Hultman<sup>28</sup>, Shaun M. Purcell<sup>30</sup>, Pamela Sklar<sup>30</sup>, Sven J. van der Lee<sup>31</sup>, Annemieke Rozemuller<sup>32</sup>, Casper Jansen<sup>32</sup>, Albert Hofman<sup>31</sup>, Robert Kraaij<sup>33</sup>, Jeroen G.J. van Rooij<sup>33</sup>, M. Arfan Ikram<sup>31</sup>, André G. Uitterlinden<sup>31,33</sup>, Cornelia M. van Duijn<sup>31</sup>, Exome Aggregation Consortium (ExAC)<sup>34</sup>, Mark J. Daly<sup>2,1</sup>, Daniel G. MacArthur†<sup>2,1</sup>

1. Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA 02142, United States
2. Analytical and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA 02114, United States
3. Program in Biological and Biomedical Sciences, Harvard Medical School, Boston, MA 02115, United States
4. Prion Alliance, Cambridge, MA 02139, United States
5. Research, 23andMe Inc., Mountain View, CA 94041, United States
6. National Prion Disease Pathology Surveillance Center, Cleveland, OH 44106, United States
7. University Hospitals Case Medical Center, Cleveland, OH 44106, United States
8. Department of Neurology and Neurobiology of Aging, Kanazawa University Graduate School of Medical Sciences, Kanazawa, Japan 920-8640
9. Department of Neurology and Neurological Science, Graduate School, Tokyo Medical and Dental University, Tokyo, Japan 113-8519
10. National Center Hospital, National Center of Neurology and Psychiatry, Tokyo, Japan 187-8551
11. Department of Public Health, Jichi Medical University, Shimotsuke, Japan 329-0498
12. Department of Neurological Science, Tohoku University Graduate School of Medicine, Sendai, Japan 980-8575
13. Australian National Creutzfeldt-Jakob Disease Registry, The University of Melbourne, Parkville, Australia 3010
14. National Creutzfeldt-Jakob Disease Research and Surveillance Unit, Western General Hospital, Edinburgh, United Kingdom EH4 2XU
15. National Reference Center for TSE, Georg-August University, Goettingen, Germany 37073
16. Center for Neuropathology and Prion Research (ZNP) at the Ludwig-Maximilians-University, Munich, Germany 81377
17. Instituto de Salud Carlos III and CIBERNED, Madrid, Spain 28031
18. Inserm U 1127, CNRS UMR 7225, Sorbonne Universités, UPMC Univ. Paris 06 UMR S 1127, Institut du Cerveau et de la Moelle épinière, ICM, 75013 Paris, France
19. Assistance Publique-Hôpitaux de Paris, Cellule Nationale de Référence des Maladies de Creutzfeldt-Jakob, Groupe Hospitalier Pitié-Salpêtrière, F-75013 Paris, France 75010
20. Assistance Publique-Hôpitaux de Paris, Service de Biochimie et Biologie moléculaire, Hôpital Lariboisière, Paris, France 75010
21. IRCCS Institute of Neurological Sciences, Bologna, Italy 40123
22. Department of Biomedical and Neuromotor Sciences, University of Bologna, Italy 40126
23. Department of Cell Biology and Neurosciences, Istituto Superiore di Sanità, Rome, Italy 00161
24. Division of Genetics and Genomics, Boston Children's Hospital, Boston, MA 02115, United States
25. Department of Biostatistics and Center for Statistical Genetics, University of Michigan School of Public Health, Ann Arbor, MI 48109, USA
26. Department of Medicine, University of Eastern Finland and Kuopio University Hospital, Kuopio, Finland 70210
27. Department of Genetics, University of North Carolina School of Medicine, Chapel Hill, NC 27599, United States
28. Karolinska Institutet, Stockholm, Sweden SE-171 77
29. Stanley Center for Psychiatric Research, Broad Institute, Cambridge, MA 02142, United States
30. Icahn School of Medicine at Mount Sinai, New York, NY 10029, United States
31. Department of Epidemiology, Erasmus MC, Rotterdam 3000 CA, The Netherlands
32. Prion Surveillance Center, Department of Pathology, University Medical Center, Utrecht 3584 CX, Netherlands
33. Department of Internal Medicine, Erasmus MC, Rotterdam 3000 CA, The Netherlands
34. A list of consortium members may be found at http://exac.broadinstitute.org/about

&dagger;Correspondence may be addressed to Eric Vallabh Minikel <eminikel@broadinstitute.org> [@cureffi](https://twitter.com/cureffi) or Daniel G. MacArthur <macarthur@atgu.mgh.harvard.edu> [@dgmacarthur](https://twitter.com/dgmacarthur)

### <a name="abstract">Abstract</a>

More than 100,000 genetic variants are reported to cause Mendelian disease in humans [[Stenson 2014], [Landrum 2014]], but the penetrance - the probability that a carrier of the purported disease-causing genotype will indeed develop the disease - is generally unknown. Here we assess the impact of variants in the prion protein gene (*PRNP*) on the risk of prion disease by analyzing 16,025 prion disease cases, 60,706 population control exomes, and 531,575 individuals genotyped by 23andMe, Inc. We show that missense variants in *PRNP* previously reported to be pathogenic are at least 30x more common in the population than expected based on genetic prion disease prevalence. While some of this excess can be attributed to benign variants falsely assigned as pathogenic, other variants have genuine effects on disease susceptibility but confer lifetime risks ranging from <0.1% to ~100%. We also show that truncating variants in *PRNP* have position-dependent effects, with true loss-of-function alleles found in healthy older individuals, supporting the safety of therapeutic suppression of prion protein expression. 

### <a name="introduction">Introduction</a>

The study of pedigrees with Mendelian disease has been tremendously successful in identifying variants that contribute to severe inherited disorders [[Brunham & Hayden 2013], [Amberger 2011], [Chong 2015]]. Causal variant discovery is enabled by selective ascertainment of affected individuals, and especially of multiplex families. Although efficient from a gene discovery perspective, the resulting ascertainment bias confounds efforts to accurately estimate the penetrance of disease-causing variants, with profound implications for genetic counseling [[Crow 1999], [Cooper 2013], [Begg 2002], [Goldwurm 2007]]. The development of large-scale genotyping and sequencing methods has recently made it tractable to perform unbiased assessments of penetrance in population controls. In several instances, such studies have suggested that previously reported Mendelian variants, as a class, are substantially less penetrant than had been believed [[Cooper 2011], [Bick 2012], [Flannick 2013], [Kirov 2014]]. To date, however, all of these studies have been limited to relatively prevalent (>0.1%) diseases, and point estimates of the penetrance of individual variants have been limited to large copy number variations [[Cooper 2011], [Kirov 2014]].

Here we demonstrate the use of large-scale population data to infer the penetrance of variants in rare, dominant, monogenic disease, using the example of prion diseases. These invariably fatal neurodegenerative disorders are caused by misfolding of the prion protein (PrP, the product of *PRNP*) [Prusiner 1998] and have an annual incidence of 1 to 2 cases per 1 million population [Klug 2013]. A small, albeit infamous, minority of cases (<1% in recent years [[CJD UK 2015], [CJD US 2015]]) are acquired through dietary or iatrogenic routes. The majority (~85%) of cases are defined as sporadic, occurring in individuals with two wild-type PRNP alleles and no known environmental exposures. Finally, ~15% of cases occur in individuals with rare, typically heterozygous, coding variants in PRNP, including missense variants, truncating variants, and octapeptide repeat insertions or deletions ([Table S1](#table_s1)). Centralized ascertainment of cases by national surveillance centers ([Materials and Methods](#methods)) makes prion disease a good test case for using reference datasets to assess the penetrance of these variants.

*PRNP* was conclusively established as a dominant disease gene due to clear Mendelian segregation of a few variants with disease [[Hsiao 1989], [Hsiao 1991b], [Medori 1992]]. Yet ascertainment bias [[Minikel 2014]], low rates of predictive genetic testing [[Owen 2014]], and frequent lack of family history [[Kovacs 2005], [Nozaki 2010]] confound attempts [[Chapman 1994], [Spudich 1995], [D'Alessandro 1998], [Mitrova & Belay 2002], [Minikel 2014]] to estimate penetrance by survival analysis. Meanwhile, the existence of non-genetic etiologies leaves doubt as to whether novel variants are causal or coincidental.

A fully penetrant disease genotype should be no more common in the population than the disease that it causes. This observation allows us to leverage two large population control datasets to re-evaluate the penetrance of reported disease variants in *PRNP*. The recently reported [Exome Aggregation Consortium](http://exac.broadinstitute.org/) (ExAC) dataset [[Lek 2015]] contains variant calls on 60,706 people ascertained for various common diseases, without any ascertainment on neurodegenerative disease. 23andMe’s database contains genotypes on 531,575 customers of its direct-to-consumer genotyping service who have opted in to participate in research, pruned to remove related individuals (first cousins or closer; [Materials and Methods](#methods)), preventing enrichment due to large families with prion disease.

### <a name="results">Results</a>

We began by asking whether reportedly pathogenic variants are as rare as expected in these population control datasets. The proportion of people alive in the population today who harbor completely penetrant variants causal for prion disease can be approximated by the product of three numbers: the annual incidence of prion disease, the proportion of cases with such a genetic variant, and the life expectancy of individuals harboring these variants. Based on upper bounds of these numbers ([Figure 1](#figure1)A), and assuming ascertainment is neutral with respect to neurodegenerative disease, we would no more than 1.7 such individuals in the 60,706 exomes in the ExAC dataset [[Lek 2015]], and ~15 such individuals among the ~530,000 genotyped 23andMe customers who opted to participate in research.

Through reviews [[Kong 2004], [Beck 2010], [Mastrianni 2010]] and PubMed searches, we identified 63 rare genetic variants reported to cause prion disease ([Table S2](#table_s2)). We reviewed ExAC read-level evidence for every rare (<0.1% allele frequency) variant call in *PRNP* ([Materials and Methods](#methods); [Table S3](#table_s3) - [S4](#table_s4)) and found that 52 individuals in ExAC harbor reportedly pathogenic missense variants ([Figure 1](#figure1)B), at least a 30-fold excess over expectation if all such variants were fully penetrant. Similarly, in the 23andMe database we observed a total of 141 alleles of 16 reportedly pathogenic variants genotyped on their platform ([Table S5](#table_s5)).

<a name="figure1"></a>

![](/figures/raster/figure1.png)

<em><b>Figure 1. Reportedly pathogenic PRNP variants are >30 times more common in controls than expected based on disease incidence.</b> Reported prion disease incidence varies with the intensity of surveillance efforts [[Klug 2013]], with an apparent upper bound of ~2 cases per million population per year (Materials and Methods). In our surveillance cohorts, 65% of cases underwent PRNP open reading frame sequencing, with 12% of all cases, or 18% of sequenced cases, possessing a rare variant ([Table S1](#table_s1)), consistent with an oft-cited estimate that 15% of cases of Creutzfeldt-Jakob disease are familial [[Masters 1979]]. Genetic prion diseases typically strike in midlife, with mean age of onset for different variants ranging from 28 to 77 [[Laplanche 1999], [Nozaki 2010], [Table S10](#table_s10)]; we accepted 80, a typical human life expectancy, as an upper bound for mean age of onset, and to be additionally conservative, we assumed that all individuals in ExAC and 23andMe were below any age of onset, even though both contain elderly individuals [[Servick 2015], [Figure S1](#figure_s1)]. Thus, no more than ~29 people per million in the general population should harbor high-penetrance prion disease-causing variants. Therefore at most ~1.7 people in ExAC (A) and ~15 people in 23andMe would be expected to harbor such variants. In fact, reportedly pathogenic variants are seen in 52 ExAC individuals (B) and on 141 alleles in the 23andMe database.</em>

Individuals with reportedly pathogenic *PRNP* variants did not cluster within any one cohort within ExAC ([Table S6](#table_s6)), arguing against enrichment due to comorbidity with a common disease ascertained for exome sequencing. ExAC does include populations, such as South Asians, in which prion disease is not closely surveilled and we cannot rule out a higher incidence than that reported in developed countries, yet the individuals with reportedly pathogenic variants in either ExAC or 23andMe were of diverse inferred ancestry ([Table S7](#table_s7), [S8](#table_s8), [S9](#table_s9)). These individuals’ ages were consistent with the overall ExAC age distribution (Figure S1), rather than being enriched below some age of disease onset. ExAC genotypes at the prion disease modifier polymorphism M129V [[Capellari 2011]] were consistent with population allele frequencies ([Table S7](#table_s7[)), rather than enriched for the lower-risk heterozygous genotype. Certain PRNP variants are associated with highly atypical phenotypes [[Moore 2001], [Mead & Reilly 2015]], which are mistakable for other dementias and may not be well ascertained by current surveillance efforts. Most of the variants found in our population control cohorts, however, have been reported in individuals with a classic, sporadic Creutzfeldt-Jakob disease phenotype [[Nozaki 2010], [Kong 2004], [Mastrianni 2010], [Zhang 2014], [Tartaglia 2010], [Peoc'h 2000]], arguing that the discrepancy between observed and expected allele counts does not result primarily from an underappreciated prevalence of atypical prion disease. 

Having observed a large excess of reportedly pathogenic variants over expectation in two datasets, and having excluded the most obvious confounders, we hypothesized that the unexpectedly high frequency of these variants in controls might arise from benign and/or low-risk variants.

We investigated which variants were responsible for the observed excess ([Figure 2](#figure2)). Variants with the strongest prior evidence of pathogenicity are absent from ExAC and cumulatively account for ≤5 alleles in 23andMe, consistent with the known rarity of genetic prion disease. Much of the excess allele frequency in population controls is due, instead, to variants with very weak prior evidence of pathogenicity ([Figure 2](#figure2) and [Supplementary Discussion](#supplementary_discusssion)). For four variants observed in controls (V180I, R208H, V210I, and M232R), pathogenicity is controversial [[Beck 2012], [Nozaki 2012]] or reduced penetrance has been suggested [[Capellari 2005], [Ripoll 1993]], but quantitative estimates of penetrance have never been produced, and the variants remain categorized as causes of genetic Creutzfeldt-Jakob disease [[Kovacs 2005], [Nozaki 2010]]. Although we cannot prove that any one of the variants we observe in population controls is completely neutral, the list of reported pathogenic variants likely includes false positives. Indeed, the observation that 0.4% (236 / 60,706) of ExAC individuals harbor a rare (<0.1%) missense variant ([Table S4](#table_s4)) suggests that ~4 of every 1000 sporadic prion disease cases will, by chance, harbor such a variant, which in many cases will be interpreted and reported as causal given the long-standing classification of *PRNP* as a Mendelian disease gene.

<a name="figure2"></a>

![](/figures/raster/figure2.png)

<em><b>Figure 2. Reportedly pathogenic *PRNP* variants include Mendelian, benign, and intermediate variants.</b> Prior evidence of pathogenicity is extremely strong for four missense variants &mdash; P102L, A117V, D178N and E200K &mdash; each of which has been observed to segregate with disease in multiple multigenerational families [[Hsiao 1989], [Goldfarb 1990], [Hsiao 1991a], [Hsiao 1991b], [Medori 1992], [Medori 1992], [Mastrianni 1995], [Webb 2008]] and to cause spontaneous disease in mouse models [[Hsiao 1990], [Dossena 2008], [Jackson 2009], [Yang 2009], [Jackson 2013], [Bouybayoune 2015]]. These account for >50% of genetic prion disease cases ([Table S1](#table_s1)), yet are absent from ExAC ([Table S3](#table_s3)), and collectively appear on ≤5 alleles in 23andMe’s cohort ([Table S5](#table_s5)), indicating allele frequencies sufficiently low to be consistent with the prevalence of genetic prion disease ([Figure 1](#figure1)). Conversely, the variants most common in controls and rare in cases had categorically weak prior evidence for pathogenicity. R208C (8 alleles in 23andMe) and P39L were observed in patients presenting clinically with other dementias, with prion disease suggested as an alternative diagnosis solely on the basis of finding a novel *PRNP* variant [[Bernardi 2014], [Zheng 2008]]. E196A was originally reported in the literature in a single patient, with a sporadic Creutzfeldt-Jakob disease phenotype and no family history [[Zhang 2014]], and appeared in only 2 of 790 Chinese prion disease patients in a recent case series [[Shi 2015]], consistent with the ~0.1% allele frequency among Chinese individuals in ExAC (Tables S5 and S8). At least three variants (M232R, V180I, and V210I) occupy a space inconsistent with either neutrality or with complete penetrance (see main text and [Figure 3](#figure3)). R148H, T188R, V203I, R208H and additional variants are discussed in [Supplementary Discussion](#supplementary_discusssion).</em>

At least three variants, however (V180I, V210I, and M232R) fail to cluster with either the likely benign or likely Mendelian variants ([Figure 2](#figure2)). Because each of these three appears primarily in one population in both cases and controls (Tables S1, S5, S7), we compared allele frequencies in matched population groups. Each has an allele frequency in controls that is too high for a fully penetrant, dominant prion disease-causing variant, and yet far lower than the corresponding allele frequency in cases ([Figure 3](#figure3)).

Because we lack genome-wide SNP data on cases we are unable to directly correct for population stratification, which thus may contribute to the observed differences in allele frequencies. Geographic clusters of genetic prion disease have been recognized for decades [[Masters 1979], [Lee 1999], [Mitrova & Belay 2002]]. For example, nearly half of Italian prion disease cases with the V210I variant are concentrated within two regions of Italy [[Ladogana 2005]], so any non-uniform geographic sampling in cases versus controls would add some uncertainty to our penetrance estimates.

Nonetheless, the magnitude of the enrichment of certain variants in cases over controls in our datasets makes substructure an implausible explanation for the entire difference. In order for V210I to be neutral and yet appear with an allele frequency of 8.1% in Italian cases despite an apparent allele frequency of 0.02% in Italian controls, it would need to be fixed in a subpopulation comprising 8% of Italy’s populace. Under this scenario, this subpopulation would need to be virtually unsampled in any of our control cohorts, and V210I cases would contain many homozygotes. In reality, no cases have been reported homozygous for this variant. Conversely, if V210I were fully penetrant, family history would be positive in most cases, and the variant’s appearance on 13 alleles in 23andMe ([Table S5](#table_s5)) would indicate that this variant alone accounts for three times the known prevalence of genetic prion disease ([Figure 1](#figure1)A). Finally, if the low family history rate were due to many *de novo* mutations, then V210I cases would be more uniformly distributed across populations ([Table S1](#table_s1)). Similar arguments rule out V180I being either benign or Mendelian. M232R, though clearly not Mendelian, could still be benign as it exhibits only 4- to 6-fold enrichment in cases, an amount which might conceivably be explained by Japanese population substructure alone. However, because even common variants in *PRNP* affect prion disease risk with odds ratios of 3 or greater [[Shibuya 1998], [Bishop 2009], [Mead 2012]], it is not implausible that M232R has a similar effect size, and our data suggest this a more likely scenario than it being neutral.

Satisfied that these three variants are likely neither benign nor Mendelian, we estimated lifetime risk in heterozygotes ([Materials and Methods](#methods)). The 1 in 1 million annual incidence of prion disease translates into a baseline lifetime risk of ~1 in 10,000 in the general population ([Materials and Methods](#methods)). Because prion diseases are so rare, even the massive enrichment of heterozygotes in cases ([Figure 3](#figure3)), implying odds ratios on the order of 10 to 1,000, corresponds to only low penetrance, with lifetime risk for M232R, V180I and V210I estimated near 0.1%, 1%, and 10%, respectively. Although our estimates are imperfect due to population stratification, they accord well with family history rates ([Figure 3](#figure3)) and explain the unique space that these variants occupy in the plot of case versus control allele count ([Figure 2](#figure2)). These data indicate that *PRNP* missense variants occupy a risk continuum rather than a dichotomy of causal versus benign.

<a name="figure3"></a>

![](/figures/raster/figure3.png)

<em><b>Figure 3. Certain variants confer intermediate amounts of lifetime risk.</b> M232R, V180I, and V210I show varying degrees of enrichment in cases over controls, indicating a weak to moderate increase in risk. Best estimates of lifetime risk in heterozygotes ([Materials and Methods](#methods)) range from ~0.08% for M232R to ~7.8% for V210I, and correlate with the likelihood of family history. Allele frequencies for P102L, A117V, D178N and E200K are consistent with up to 100% penetrance, with confidence intervals including all reported estimates of E200K penetrance based on survival analysis, which range from ~60% to ~90% [[Chapman 1994], [Spudich 1995], [D'Alessandro 1998], [Mitrova 2002], [Minikel 2014]]. Rates of family history of neurodegenerative disease in Japanese cases are from ([Table S10](#table_s10)) and in European populations are from [[Kovacs 2005]], with Wilson binomial confidence intervals shown. \*Based on allele counts rounded for privacy ([Materials and Methods](#methods)). &dagger;GSS, Gerstmann Straussler Scheinker disease associated with variants P102L, A117V and G131V. &Dagger;FFI: fatal familial insomnia associated with a D178N *cis* 129M haplotype.</em>

We asked whether the same was true of protein-truncating variants. PRNP possesses only one protein-coding exon, so premature stop codons are expected to result in truncated polypeptides rather than in nonsense-mediated decay. Prion diseases are known to arise from a gain of function, as neurodegeneration is not seen in mice, cows, or goats lacking PrP [[Bueler 1992], [Richt 2008], [Yu 2009], [Benestad 2012]], and the rate of prion disease progression is tightly correlated with PrP expression level [[Fischer 1996]]. Yet heterozygous C-terminal (residue ≥145) truncating variants are known to cause prion disease, sometimes with peripheral amyloidosis [[Mead & Reilly 2015]]. These patients also experience sensorimotor neuropathy phenotypically similar to that present in homozygous, but not heterozygous, PrP knockout mice [[Bremer 2010]], but attributed to amyloid infiltration of peripheral nerves, rather than loss of PrP function [[Mead & Reilly 2015]].

We identified, for the first time, heterozygous N-terminal (residue ≤131) truncating variants in four ExAC individuals and were able to obtain Sanger validation ([Figure S2](#figure_s2)) and limited phenotype data ([Table S11](#table_s11)) for three. These individuals are free of overt neurological disease at ages 79, 73, and 52, and report no personal or family history of neurodegeneration nor of peripheral neuropathy. Therefore, the pathogenicity of protein-truncating variants appears to be dictated by position within PrP’s amino acid sequence ([Figure 4](#figure4)). Observing three *PRNP* nonsense variants in ExAC is consistent with the expected number (~3.9) once we adjust our model [[Samocha 2014]] to exclude codons ≥145, where truncations cause a dominant gain-of-function disease. Thus, we see no evidence that *PRNP* is constrained against truncation in its N terminus. This, combined with the lack of any obvious phenotype in individuals with N-terminal truncating variants, suggests that heterozygous loss of PrP function is tolerated.

<a name="figure4"></a>

![](/figures/raster/figure4.png)

<em><b>Figure 4. Effects of truncating variants in the human prion protein are position-dependent.</b> Truncating variants reported in prion disease cases in the literature ([Table S2](#table_s2)) and in our cohorts ([Table S1](#table_s1)) cluster exclusively in the C-terminal region (residue ≥145), while truncating variants in ExAC are more N-terminal (residue ≤131). The ortholog of each residue from 23-94 is deleted in at least one prion-susceptible transgenic mouse line [[Aguzzi 2008]]. C-terminal truncations abolish PrP’s glycosylphosphatidylinositol anchor but leave most of the protein intact, a combination that mediates gain of function through mislocalization, causing this normally cell-surface-anchored protein to be secreted. Consistent with this model of pathogenicity, mice expressing full-length secreted PrP develop fatal and transmissible prion disease [[Chesebro 2010], [Stohr 2011]]. By contrast, the N-terminal truncating variants that we observe retain only residues dispensable for prion propagation, and are likely to cause a total loss of protein function.</em>

### <a name="discussion">Discussion</a>

Over 100,000 genetic variants have been reported to cause Mendelian disease in humans [[Stenson 2014], [Landrum 2014]]. Many such reports do not meet current standards for assertions of pathogenicity [[MacArthur 2014], [Richards 2015]], and if all such reports were believed, the cumulative frequency of these variants in the population would imply that most people have a genetic disease [[Lek 2015]]. It is generally unclear how much of the excess burden of purported disease variants in the population is due to benign variants falsely associated, and how much is due to variants with genuine association but incomplete penetrance.

Here we leverage newly available large genomic reference datasets to re-evaluate reported disease associations in a dominant disease gene, PRNP. We identify some missense variants as likely benign while showing that others span a spectrum from <0.1% to ~100% penetrance. Our analyses provide quantitative estimates of lifetime risk for hundreds of asymptomatic individuals who have inherited incompletely penetrant PRNP variants.

Available datasets are only now approaching the size and quality required for such analyses, resulting in limitations for our study. The confidence intervals on our lifetime risk estimates span more than an order of magnitude, and our inability to perfectly control for population stratification injects additional uncertainty. We have been unable to reclassify those PRNP variants that are very rare both in cases and in controls ([Supplementary Discussion](#supplementary_discusssion)). We have avoided analysis of large insertions that are poorly called with short sequencing reads, though we note that existing literature on these insertions is consistent with a spectrum of penetrance similar to that which we observe for missense variants [[Kong 2004], [Mead 2006b]]. Penetrance estimation in Mendelian disease will be improved by the collection of larger case series, particularly with genome-wide SNP data to allow more accurate population matching. This, coupled with continued large-scale population control sequencing and genotyping efforts, should reveal whether the dramatic variation in penetrance that we observe here is a more general feature of dominant disease genes.

Because PrP is required for prion pathogenesis and reduction in gene dosage slows disease progression [[Bueler 1993], [Fischer 1996], [Mallucci 2003], [Safar 2005]], several groups have sought to therapeutically reduce PrP expression using RNAi [[White 2008], [Pulford 2010], [Ahn 2014]], antisense oligonucleotides [[Nazor Friberg 2012]], or small molecules [[Karapetyan & Sferrazza 2013], [Silber 2014]]. Our discovery of heterozygous loss-of-function variants in three healthy older humans provides the first human genetic data regarding the effects of a 50% reduction in gene dosage for PRNP. Both the number of individuals and the depth of available phenotype data are limited, and lifelong heterozygous inactivation of a gene is an imperfect model of the effects of pharmacological depletion of the gene product. With those limitations, our data provide preliminary evidence that a reduction in PRNP dosage, if achievable in patients, is likely to be tolerated. Increasingly large control sequencing datasets will soon enable testing whether the same is true of other genes currently being targeted in substrate reduction therapeutic approaches for other protein-folding disorders.

Together, our findings highlight the value of large reference datasets of human genetic variation for informing both genetic counseling and therapeutic strategy. 

### <a name="methods">Materials and methods</a>

#### Prion disease case series

Prion disease is considered a notifiable diagnosis in most developed countries, with mandatory reporting of all suspect cases to a centralized surveillance center. Surveillance was carried out broadly according to established guidelines [[WHO 1998](http://www.who.int/csr/resources/publications/bse/whoemczdi989.pdf), [WHO 2003](http://www.who.int/bloodproducts/TSE-manual2003.pdf)], with specifics as described previously for Australia [[Collins 2002]], France [[Brandel 2011]], Germany [[Windl 1999], [Grasbon-Frodl 2004], [Zerr 2009]], Italy [[Puopolo 2003]], Japan [[Nozaki 2010]], and the Netherlands [[Jansen 2012]]. Sanger sequencing of the *PRNP* open reading frame was performed as described [[Parchi 1999]]. We included only prion disease cases classified as definite (autopsy-confirmed) or probable according to published guidelines [[WHO 2003](http://www.who.int/bloodproducts/TSE-manual2003.pdf)]. Criteria for genetic testing vary between countries and over the years of data collection, with testing offered only on indication of family history in some times and places, and testing of all suspect cases with tissue available in other instances. Summary statistics on the total number and proportion of cases sequenced are presented in [Table S1](#table_s1).

#### Exome sequencing and analysis

The ascertainment, sequencing, and joint calling of the ExAC dataset have been described previously [[Lek 2015]]. We extracted all rare (<0.1%) coding variant calls in *PRNP* with genotype quality (GQ) ≥10, alternate allele depth (AD) ≥3 and alternate allele balance (AB) ≥20%. Read-level evidence was visualized using Integrative Genomics Viewer (IGV) [[Robinson 2011]] for manual review. Because most ExAC exomes were sequenced with 76bp reads and the *PRNP* octapeptide repeat region (codons 50-90 inclusive) is 123bp long, it was impossible to determine whether genotype calls in this region were correct, and they were not considered further. After review of IGV screenshots, 87% of genotype calls were judged to be correct and were included in [Table S3](#table_s3). Of the genotype calls judged to be correct, 99% had genotype quality (GQ) ≥95, 99% had allelic balance (AB) between 30% and 70%, and 97% had ≥10 reads supporting the alternate allele.

All participants provided informed consent for exome sequencing and analysis. The Exome Aggregation Consortium’s aggregation and release of exome data have been approved by the Partners Healthcare Institutional Research Board (2013P001339). ExAC data have been publicly released at [http://exac.broadinstitute.org/](http://exac.broadinstitute.org/) and IGV screenshots of the rare *PRNP* variants deemed to be genuine and included in this study are available at [https://github.com/ericminikel/prnp_penetrance/tree/master/supplement/igv](https://github.com/ericminikel/prnp_penetrance/tree/master/supplement/igv) 

#### 23andMe research participants and genotyping

Participants were drawn from the customer base of 23andMe, Inc., a personal genetics company (accessed February 6, 2015). All participants provided informed consent under a protocol approved by an external AAHRPP-accredited IRB, Ethical & Independent Review Services (E&I Review). DNA extraction and genotyping were performed on saliva samples by National Genetics Institute (NGI), a CLIA-licensed clinical laboratory and a subsidiary of Laboratory Corporation of America.  Samples were genotyped on one of four Illumina platforms (V1-V4) as described previously [[Bryc 2015]].  Of the *PRNP* SNPs considered, two (P105L and E200K) were genotyped on all four platforms while the other 14 were genotyped only on V3 and V4, resulting in differing numbers of total samples genotyped ([Table S5](#table_s5)). Genotypes were called with Illumina GenomeStudio.  A 98.5% call rate were required for all samples.  As with all 23andMe research participants, individuals whose genotyping analyses failed to reach the desired call rate repeatedly were recontacted to provide additional samples.  A maximal set of unrelated individuals was chosen based on segmental identity-by-descent (IBD) estimation [[Durand 2014a]].  Individuals were defined as related if they shared more than 700 cM IBD (approximately the minimal expected sharing between first cousins). Allele counts between 1 and 5 were rounded up to 5 to protect individual privacy ([Table S5](#table_s5)). Rounding down to 1 instead would raise our estimates of penetrance for V180I to 7.7% (95%CI, 1.2% - 50%) and for P102L, A117V, D178N and E200K collectively to 100% (95%CI, 100% - 100%), but the confidence intervals would still overlap those based on ExAC allele frequencies, and the overall conclusions of our study would remain unchanged.

#### 23andMe ancestry composition

Ancestral origins of chromosomal segments were assigned on a continental level (European, Latino, African, and East Asian) and a country level (Japanese) as described by Durand et al [[Durand 2014b]].  Briefly, after phasing genotypes using an out-of-sample implementation of the Beagle algorithm [[Browning & Browning 2007]], a string kernel support vector machine classifier assigns tentative ancestry labels to local genomic regions.  Then an autoregressive pair hidden Markov model was used to simultaneously correct phasing errors and produce reconciled local ancestry estimates and confidence scores based on the initial assignment. Finally, isotonic regression models were used to recalibrate the confidence estimates.

Europeans and East Asians were defined as individuals with more than 97% of chromosomal segments predicted as being from the respective ancestries.  Because African Americans and Latinos are highly admixed, no single threshold of genome-wide ancestry is sufficient to distinguish them.  However, segment length distributions of European, African, and Native American ancestries are different between African Americans and Latinos, due to distinct admixture timing in the two ethnic groups. Thus, a logistic classifier based on segment length of European, African, and Native American ancestries was used to distinguish between African Americans and Latinos. 

At the country level, individuals were classified as Japanese based on the fraction of the respective local ancestry using a threshold of 90% for classifying Japanese ancestry. This threshold is based on the average fraction of local ancestry in the reference population (23andMe research participants with all four grandparents from the reference country): 94% (5% SD, *N*=533) for Japanese. Using the same approach, we were unable to obtain a confident set of Italian individuals for analysis of V210I due to extensive admixture. 23andMe research participants with all four grandparents from Italy only have 66% (18% SD, *N*=2090) Italian ancestry, and only ~60 participants have >90% Italian ancestry.

#### ExAC ancestry inference

We computed ten principal components based on ~5,800 common SNPs as described [[Purcell 2014], [Lek 2015]]. A centroid in eigenvalue-weighted principal component space was generated for each HapMap population based on 1000 Genomes individuals in ExAC. The remaining individuals in ExAC were assigned to the HapMap population with the nearest centroid according to eigenvalue-weighted Euclidean distance. Ancestries of all individuals, including those with reportedly pathogenic variants, are summarized in (Tables S7, S8). 

#### Prion disease incidence and baseline risk

The reported incidence of prion disease varies between countries and between years, with much of the variability explained by the intensity of surveillance, as measured by the number of cases referred to national surveillance centers [[Klug 2013]]. Rates of ~1 case per million population per year have been reported, for instance in the U.S. [[Holman 2010]] and in Japan [[Nozaki 2010]], however, the countries with the most intense surveillance (greatest number of referrals per capita), such as France and Austria, observe incidence figures as high as 2 cases per million population per year [[Klug 2013]]. Only in small countries where the statistics are dominated by a particular genetic prion disease founder mutation, such as Israel and Slovakia [[Chapman 1994], [Mitrova & Belay 2002]], has an incidence higher than 2 per million been consistently observed [[EUROCJD Surveillance Data]]. We therefore accepted 2 cases per million as an upper bound for the true incidence of prion disease. Assuming an all-causes death rate of ~10 per 1,000 annually [[UN Population and Vital Statistics Report]], this incidence corresponds to prion disease accounting for ~0.02% of all deaths, which we accepted as the baseline disease risk in the general population.  

#### Lifetime risk estimation

By Bayes' theorem, the probability of disease given a genotype (penetrance or lifetime risk, P(D|G)) is equal to the proportion of individuals with the disease who have the genotype (genotype frequency in cases, P(G|D)) times the prevalence of the disease (baseline lifetime risk in the general population, P(D)), divided by the frequency of the genotype in the general population (here, population control allele frequency, P(G)). The use of this formula to estimate disease risk dates back at least to Cornfield's estimation of the probability of lung cancer in smokers [[Cornfield 1951]], with later contributions by Woolf [[Woolf 1955]] and a synthesis by C.C. Li with application to genetics [[Li 1961]]. 

We used an allelic rather than genotypic model, such that lifetime risk in an individual with one allele is equal to case allele frequency (based on the number of prion disease cases that underwent PRNP sequencing) times baseline risk divided by population control allele frequency, P(D|A) = P(A|D)&times;P(D)/P(A). Note that we assume that our population control datasets include individuals who will later die of prion disease, thus enabling direct use of the ExAC and 23andMe allele frequencies as the denominator P(A). Following Kirov [[Kirov 2014]], we  compute Wilson 95% confidence intervals on the binomial proportions P(A|D) and P(A), and calculate the upper bound of the 95% confidence interval for penetrance using the upper bound on case allele frequency and the lower bound on population control allele frequency, and vice versa for the lower bound on penetrance. 

#### Source code availability

Data processing, analysis, and figure generation utilized custom scripts written in Python 2.7.6 and R 3.1.2. These scripts, along with vector graphics of all figures and tab-delimited text versions of all supplementary tables, are available online at 
[https://github.com/ericminikel/prnp_penetrance](https://github.com/ericminikel/prnp_penetrance).

### <a name="acknowledgments">Acknowledgments</a>

#### Funding

Research reported in this publication was partially supported by the National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK) and the National Institute of General Medical Sciences (NIGMS) of the National Institutes of Health, under awards U54DK105566 and R01GM104371, by Broad Institute NextGen funds, and by Prion Alliance sundry funds. Sonia Vallabh is supported by the National Science Foundation (NSF) Graduate Research Fellowship Program (GRFP) grant number 2015214731. U.S. prion surveillance work was conducted under Centers for Disease Control and Prevention (CDC) contract UR8/CCU515004. Japanese prion surveillance work was supported by a grant-in-aid from the Research Committee of Prion Disease and Slow Virus Infection, the Ministry of Health, Labour and Welfare of Japan, and from the Research Committee of Surveillance and Infection Control of Prion Disease, the Ministry of Health, Labour and Welfare of Japan. The French surveillance network is supported by the Institut National de veille Sanitaire. German prion surveillance work was supported by Robert Koch-Institute / Federal Ministry of Health grant 1369-341. The UK National CJD Research and Surveillance Unit is supported by the Department of Health and the Scottish Executive. The Australian National Creutzfeldt-Jakob Disease Registry is funded by the Commonwealth Department of Health. SJC is supported by a NHMRC Practitioner Fellowship: identification #APP1005816. Contributions at Erasmus MC were supported by Netherlands Genomics Initiative (NGI)/Netherlands Organisation for Scientific Research (NWO) sponsored Netherlands Consortium for Healthy Aging (NCHA; project 050-060-810), by the Genetic Laboratory of the Department of Internal Medicine, Erasmus MC, by a Complementation Project of the Biobanking and Biomolecular Research Infrastructure Netherlands (BBMRI-NL; www.bbmri.nl ; project number CP2010-41), by Erasmus Medical Center and Erasmus University, Rotterdam, Netherlands Organization for the Health Research and Development (ZonMw), the Research Institute for Diseases in the Elderly (RIDE), the Ministry of Education, Culture and Science, the Ministry for Health, Welfare and Sports, the European Commission (DG XII), and the Municipality of Rotterdam. We thank the customers of 23andMe, ExAC research participants, and prion disease patients and families who participated in this research.

#### Author contributions

E.V.M., S.M.V. and D.G.M. conceived and designed the study. E.V.M analyzed data, generated figures, and wrote the manuscript. S.M.V. and E.V.M. reviewed literature and IGV screenshots. K.E.S. performed constraint analyses. M.Lek, K.E., K.E.S., K.J.K., A.H.O.-L., M.J.D., and D.G.M. consulted on data analysis and interpretation. J.F.S., C.Y.M., J.Y.C., and L.P.C.Y. prepared and consulted on analysis of 23andMe data. P.G., J.B., S.Z., Y.C., W.C., M.Y., T.H., N.S., H.M., Y.N., T.K., S.J.C., A.B., R.G.W., R.Knight, C.P., I.Z., T.F.J.K., S.E., A.G., M.C., J.d.P.C., S.H., J.-L.L., E.B.-A., J.-P.B., S.C, P.P., A.L., A.P., R.Kraaij., J.G.J.v.R., S.J.v.d.L., R.M., and C.v.D. prepared and consulted on analysis of prion surveillance data. E.V.M., J.L.M., M.B., M.Laakso, K.M., A.K., K.C., S.A.M., P.S., C.M.H., S.M.P., P.S., C.v.D., F.R.R., A.H., A.I., S.J.v.d.L., J.M.V.-D., and A.G.U. prepared and consulted on analysis of data regarding protein-truncating variants. ExAC provided exome sequence data.

### <a name="supplement">Supplemental information</a>

Table of contents:

+ [Supplementary Discussion](#supplementary_discusssion)
    + Additional variants
    + Dominant versus allelic models
+ [Table S1](#table_s1). Allele counts of rare *PRNP* variants in 16,025 definite and probable prion disease cases in 9 countries
+ [Table S2](#table_s2). Rare *PRNP* variants reported in peer-reviewed literature to cause prion disease
+ [Table S3](#table_s3). Allele counts of rare *PRNP* variants in 60,706 individuals in ExAC
+ [Table S4](#table_s4). Summary of rare *PRNP* variants by functional class in ExAC
+ [Table S5](#table_s5). Allele counts of 16 reportedly pathogenic *PRNP* variants in >500,000 23andMe research participants
+ [Table S6](#table_s6). Phenotypes investigated in studies in which ExAC individuals with reportedly pathogenic *PRNP* variants were ascertained
+ [Table S7](#table_s7). Inferred ancestry and codon 129 genotypes of ExAC individuals with reportedly pathogenic variants
+ [Table S8](#table_s8). Inferred ancestry of all ExAC individuals
+ [Table S9](#table_s9). Inferred ancestry of 23andMe research participants
+ [Table S10](#table_s10). Details of Japanese prion disease cases
+ [Table S11](#table_s11). Phenotypes of individuals with N-terminal PrP truncating variants
+ [Figure S1](#figure_s1). Age of ExAC individuals with reportedly pathogenic *PRNP* variants versus all individuals in ExAC
+ [Figure S2](#figure_s2). Sanger sequencing results for individuals with N-terminal truncating variants

#### <a name="supplementary_discussion">Supplementary discussion</a>

**Additional variants**

Of the 63 reportedly pathogenic variants ([Table S2](#table_s2)), 10 are discussed in the main text. Of those 10, our data and our analysis of the literature indicate high penetrance for 4 (P102L, A117V, D178N, and E200K), intermediate penetrance for 3 (V180I, V210I, and M232R), and suggest that 3 others may be benign (P39L, E196A, and R208C). In this section we discuss four additional variants that we cannot conclusively reclassify but which are unlikely to be highly penetrant, and we also provide a brief discussion of interpretation for remaining variants.

+ **R148H** has been reported in a two isolated patients with a sporadic Creutzfeldt-Jakob disease phenotype and negative family history [[Krebs 2005], [Pastore 2005]] and appears one additional time in our case cohorts (Table S1). Based on its rarity in cases, lack of familial segregation and presence on 3 alleles in ExAC, it is unlikely to be a highly penetrant Mendelian variant. It might be benign or might slightly increase prion disease risk.
+ **T188R** has been reported in two cases in the literature. One German individual presented with a sporadic Creutzfeldt-Jakob disease phenotype but no autopsy was performed; family history was negative [[Windl 1999], [Roeber 2008]]. One Mexican-American individual had autopsy-confirmed prion disease and an ambiguous family history [[Tartaglia 2010]]. This variant appears 12 times in our case cohort (all in the United States) and 3 times in ExAC (all in Latino populations). Based on its allele frequency in controls, rarity in cases and lack of any clear evidence for segregation in families, T188R is unlikely to be a highly penetrant Mendelian disease variant. It is not clear whether it is benign or increases prion disease risk.
+ **V203I** has been reported in three heterozygous patients - one Italian [[Peoc'h 2000]], one Korean [[Jeong 2010]], and one Chinese [[Shi 2013]], as well as in one Japanese homozygote [[Komatsu 2014]]. Family history is negative in all of these reported patients as well as in two additional V203I cases in our Japanese case cohort ([Table S10](#table_s10)). In our cohorts, this variant appears in a total of 16 cases from several countries; in ExAC, it appears in 3 European individuals. Based on its allele frequency in controls, rarity in cases and lack of any clear evidence for segregation in families, V203I is unlikely to be a highly penetrant Mendelian disease variant, and could be benign or could increase prion disease risk. The report of prion disease in a V203I homozygote makes us slightly inclined to favor the interpretation that V203I does increase prion disease risk.
+ **R208H** has been reported in several isolated cases of varied ancestries, all with a negative family history [[Mastrianni 1996], [Capellari 2005], [Roeber 2005], [Basset-Leobon 2006], [Chen 2011], [Matej 2012], [Vita 2013]]. In our cohorts, it appears in 13 prion disease cases, 9 ExAC individuals and 22 individuals in the 23andMe database. Given its high frequency in controls, this variant may be benign or may slightly increase prion disease risk.
+ **Other variants.** Excluding variants discussed in the main text and above, 0.8% (87 / 10460) of individuals in our case series harbor other rare *PRNP* missense variants, some of which have been reported as pathogenic ([Table S2](#table_s2)) and others of which have not. Because most of these variants are very rare both in cases and in population controls, comparisons of case and control allele frequency are not well powered to evaluate the pathogenicity of most individual variants. we are unable to reach any firm conclusions about their pathogenicity. Collectively, our data indicate that this category includes at least some variants that increase prion disease risk, because only 0.3% (187 / 60706) of ExAC individuals harbor a rare missense variant other than those discussed in the main text or above, whereas 0.8% (87 / 10460) of prion disease cases harbor one of these variants, a significant enrichment (p = 1 &times; 10<sup>-12</sup>, Fisher's exact test). Indeed, Mendelian segregation has been demonstrated for some of these variants, such as T183A and F198S [[Nitrini 1997], [Hsiao 1992]]. However, the fact that, in the aggregate, we observe only modest (~3-fold) enrichment of such variants in cases versus controls suggests that this category also includes many neutral or very low-risk variants, consistent with our expectation that sporadic prion disease cases should, by chance, harbor some rare variants unassociated with disease. We also cannot exclude the possibility that some specific rare variants, particularly those observed in controls and not in cases, could be protective. 
+ **Future novel missense variants.** Additional novel missense variants in *PRNP* are sure to be observed in prion disease patients in the future. Our findings that some reportedly pathogenic variants are either benign or exhibit low penetrance, together with our observation that ~4 in 1000 controls harbor a rare *PRNP* missense variant, urge caution in the interpretation of novel variants in prion disease patients. This is consistent with current guidelines [[MacArthur 2014], [Richards 2015]], which indicate that novel protein-altering variants, even in established disease genes, should not be assumed to be causal or highly penetrant until evidence, such as Mendelian segregation, or significant enrichment in cases over controls, can be established.

**Dominant versus allelic models**

Virtually all patients ever reported with genetic prion disease have been heterozygous for the putative pathogenic variants. Five individuals homozygous for E200K [[Simon 2000]] were reported to have a younger age of onset than heterozygotes (mean 50 vs. 59 years, p = .03), suggesting some degree of codominance. There have been individual case reports of homozygotes for Q212P [[Beck 2010]] and V203I [[Komatsu 2014]], both without a family history among heterozygote relatives, which might suggest that dosage of the mutant allele is important. We are not aware of any other reports of individuals homozygous for potentially pathogenic variants in *PRNP*. Regardless of whether a dominant or allelic model is assumed, our formula for lifetime risk ([Materials and Methods](#methods)) gives identical point estimates of penetrance and virtually identical 95% confidence intervals.

#### <a name="table_s1">Table S1</a>. Allele counts of rare *PRNP* variants in 16,025 definite and probable prion disease cases in 9 countries. 

Abbreviations: OPRD, octapeptide repeat deletion; OPRI, octapeptide repeat insertion. 

\*V203I in Japan: two heterozygotes and one homozygote, four alleles total. All other individuals are heterozygotes.

| country | Australia | France | Germany | Italy | Japan | Netherlands | Spain | U.K. | U.S. | TOTAL |
| ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | 
| Start year | 1993 | 1991 | 1993 | 1993 | 1999 | 1993 | 1993 | 1990 | 2000 |  |
| End year | 2014 | 2013 | 2015 | 2013 | 2014 | 2013 | 2013 | 2013 | 2014 |  |
| Definite plus probable cases | 553 | 2383 | 2690 | 1684 | 2144 | 409 | 1280 | 1963 | 2919 | 16025 |
| Of which *PRNP* sequenced | 152 | 1774 | 1307 | 1054 | 1533 | 163 | 749 | 1088 | 2640 | 10460 |
| Proportion sequenced | 27% | 74% | 49% | 63% | 72% | 40% | 59% | 55% | 90% | 65% |
| Number with rare variants | 31 | 196 | 125 | 396 | 464 | 22 | 127 | 173 | 361 | 1895 |
| Proportion with rare variants | 6% | 8% | 5% | 24% | 22% | 5% | 10% | 9% | 12% | 12% |
| 2-OPRD |  |  |  |  |  | 3 |  |  |  | 3 |
| 1-OPRI |  | 2 | 1 |  |  |  |  |  | 4 | 7 |
| 2-OPRI |  |  |  |  |  |  | 1 |  | 5 | 6 |
| 3-OPRI |  | 1 | 1 |  |  |  |  |  |  | 2 |
| 4-OPRI |  | 1 | 3 |  |  |  | 2 | 13 | 4 | 23 |
| 5-OPRI |  | 2 | 10 |  |  | 1 | 1 | 13 | 12 | 39 |
| 6-OPRI |  | 2 |  |  |  |  |  | 35 | 15 | 52 |
| 7-OPRI |  | 1 | 1 |  |  | 1 |  | 2 |  | 5 |
| 8-OPRI |  | 10 |  |  |  |  |  |  |  | 10 |
| 9-OPRI |  |  |  |  |  |  |  |  | 4 | 4 |
| 10-OPRI |  |  |  |  |  |  |  | 1 |  | 1 |
| OPRI (length unspecified) |  |  |  | 9 | 8 |  |  |  |  | 17 |
| A2V |  |  | 1 |  |  |  |  |  |  | 1 |
| G54S |  |  |  |  |  |  |  | 1 | 4 | 5 |
| P84S |  |  |  |  |  |  |  | 1 |  | 1 |
| G88A |  |  |  |  |  |  | 1 |  |  | 1 |
| G94S |  |  |  |  |  |  |  |  | 1 | 1 |
| H96Y |  |  |  |  |  |  | 1 |  |  | 1 |
| P102L | 2 | 10 | 7 | 59 | 83 |  | 1 | 34 | 25 | 221 |
| P105L |  |  |  |  | 12 |  |  | 1 |  | 13 |
| P105S |  |  |  |  |  |  |  |  | 1 | 1 |
| P105T | 3 |  | 2 |  |  |  |  |  |  | 5 |
| G114V |  |  |  |  |  |  |  |  | 1 | 1 |
| A117V |  | 3 |  |  |  | 8 | 1 | 12 | 9 | 33 |
| G131V |  |  |  |  |  | 1 |  |  |  | 1 |
| S132I |  |  |  |  |  |  |  | 1 |  | 1 |
| A133V | 1 |  |  |  |  |  |  |  | 1 | 2 |
| R148H |  |  | 1 |  |  |  |  |  | 2 | 3 |
| Q160X |  |  |  |  |  |  |  |  | 1 | 1 |
| Y163X |  |  |  |  |  |  |  | 2 |  | 2 |
| D167G |  |  |  |  |  |  |  | 1 |  | 1 |
| V176G | 1 |  |  |  |  |  |  |  |  | 1 |
| D178N | 3 | 34 | 32 | 18 | 5 | 4 | 65 | 12 | 36 | 209 |
| V180I |  | 1 |  | 1 | 218 |  |  |  | 5 | 225 |
| T183A |  |  |  |  |  |  |  |  | 3 | 3 |
| Q186X |  |  |  |  |  |  |  |  | 1 | 1 |
| H187A |  |  |  |  |  |  |  |  | 1 | 1 |
| H187R |  |  |  |  |  |  |  |  | 7 | 7 |
| T188A | 1 |  |  |  |  |  |  |  |  | 1 |
| T188K |  |  | 2 |  |  |  |  |  | 1 | 3 |
| T188R |  |  |  |  |  |  |  |  | 12 | 12 |
| E196A |  |  |  |  |  |  |  |  | 1 | 1 |
| E196K |  | 3 | 8 | 2 |  |  |  |  |  | 13 |
| F198S |  |  |  |  |  |  |  |  | 5 | 5 |
| E200G |  |  |  |  |  |  |  |  | 1 | 1 |
| E200K | 11 | 101 | 28 | 123 | 63 | 2 | 52 | 38 | 153 | 571 |
| V203I |  | 5 |  | 3 | 4 |  |  |  | 5 | 17 |
| R208H |  | 1 | 2 | 7 | 1 |  |  |  | 4 | 15 |
| V210I | 4 | 13 | 19 | 171 | 1 |  |  | 3 | 36 | 247 |
| E211Q |  | 5 | 2 | 3 |  |  |  | 1 |  | 11 |
| E211D |  | 1 |  |  |  |  |  |  |  | 1 |
| Q212P |  |  |  |  |  |  |  | 2 |  | 2 |
| I215V |  |  |  |  |  |  | 1 |  |  | 1 |
| Y218N |  |  |  |  |  |  | 1 |  |  | 1 |
| A224V |  |  |  |  |  |  |  |  | 1 | 1 |
| Y226X |  |  |  |  |  | 1 |  |  |  | 1 |
| Q227X |  |  |  |  |  | 1 |  |  |  | 1 |
| M232R |  |  |  |  | 63 |  |  |  |  | 63 |
| V180I and M232R in *trans* |  |  |  |  | 4 |  |  |  |  | 4 |
| Variant not specified | 5 |  | 5 |  | 2 |  |  |  |  | 12 |

#### <a name="table_s2">Table S2</a>. Rare *PRNP* variants reported in peer-reviewed literature to cause prion disease

*Note: an updated version of this table is maintained in [this blog post](http://www.cureffi.org/2015/01/13/list-of-reportedly-pathogenic-prnp-variants/).*

| variant | first report | see also |
| ---- | ---- | ---- |
| P39L | [Bernardi 2014] |  |
| 2-OPRD | [Beck 2001] | [Capellari 2002] |
| 1-OPRI | [Laplanche 1995] | [Pietrini 2003] |
| 2-OPRI | [Hill 2006] |  |
| 3-OPRI | [Nishida 2004] |  |
| 4-OPRI | [Laplanche 1995] | [Campbell 1996], [Kaski 2011] |
| 5-OPRI | [Goldfarb 1991b] |  |
| 6-OPRI | [Owen 1990] | [Mead 2006b] |
| 7-OPRI | [Goldfarb 1991b] | [Lewis 2003] |
| 8-OPRI | [Goldfarb 1991b] | [Laplanche 1999] |
| 9-OPRI | [Krasemann 1995] |  |
| 12-OPRI | [Kumar 2011] |  |
| P84S | [Jones 2014] |  |
| S97N | [Zheng 2008] |  |
| P102L | [Goldgaber 1989] | [Hsiao 1992] |
| P105L | [Yamada 1993] | [Yamada 1999] |
| P105S | [Tunnell 2008] |  |
| P105T | [Rogaeva 2006] | [Polymenidou 2011] |
| G114V | [Rodriguez 2005] | [Liu 2010] |
| A117V | [Tateishi 1990] | [Hsiao 1991a] |
| 129insLGGLGGYV | [Hinnell 2011] |  |
| G131V | [Panegyres 2001] | [Jansen 2012] |
| S132I | [Hilton 2009] |  |
| A133V | [Rowe 2007] |  |
| Y145X | [Kitamoto 1993] |  |
| R148H | [Krebs 2005] | [Pastore 2005] |
| Q160X | [Finckh 2000] | [Jayadev 2011] |
| Y163X | [Revesz 2009] | [Mead 2013] |
| D167G | [Bishop 2009] |  |
| D167N | [Beck 2010] |  |
| V176G | [Simpson 2013] |  |
| D178Efs25X | [Matsuzono 2013] |  |
| D178N | [Goldfarb 1991a] | [Medori 1992], [Goldfarb 1992] |
| V180I | [Hitoshi 1993] | [Chasseigneaux 2006] |
| T183A | [Nitrini 1997] | [Grasbon-Frodl 2004] |
| H187R | [Butefisch 2000] |  |
| T188A | [Collins 2000] |  |
| T188K | [Finckh 2000] | [Roeber 2008] |
| T188R | [Windl 1999] | [Roeber 2008], [Tartaglia 2010] |
| T193I | [Kotta 2006] |  |
| E196A | [Zhang 2014] |  |
| E196K | [Peoc'h 2000] |  |
| F198S | [Farlow 1989] | [Hsiao 1992] |
| F198V | [Zheng 2008] |  |
| E200G | [Kim 2013] |  |
| E200K | [Goldgaber 1989] | [Hsiao 1991b] |
| D202G | [Heinemann 2008] |  |
| D202N | [Piccardo 1998] |  |
| V203I | [Peoc'h 2000] |  |
| R208C | [Zheng 2008] |  |
| R208H | [Mastrianni 1996] | [Capellari 2005], [Roeber 2005] |
| V210I | [Ripoll 1993] | [Pocchiari 1993], [Mouillet-Richard 1999] |
| E211D | [Peoc'h 2012] |  |
| E211Q | [Peoc'h 2000] |  |
| Q212P | [Piccardo 1998] |  |
| I215V | [Munoz-Nieto 2013] |  |
| Q217R | [Hsiao 1992] |  |
| Y218N | [Alzualde 2010] |  |
| Y226X | [Jansen 2010] |  |
| Q227X | [Jansen 2010] |  |
| M232R | [Hitoshi 1993] | [Hoque 1996] |
| M232T | [Bratosiewicz 2000] |  |
| P238S | [Windl 1999] |  |

#### <a name="table_s3">Table S3</a>. Allele counts of rare *PRNP* variants in 60,706 individuals in ExAC. 

Chromosomal positions are given in GRCh37 coordinates and HGVS notations are given relative to Ensembl transcript ENST00000379440. Mean read depth across the *PRNP* coding sequence was 55.21. Call rate is the proportion of ExAC individuals with a genotype call of genotype quality (GQ) ≥20 and a depth (DP) of ≥10 reads.

| Chrom | Pos | Ref | Alt | HGVS | Variant | Class | Call rate | AC |
| ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- |
| 20 | 4679863 | C | T | c.-4C>T |  | non-coding | 97% | 1 |
| 20 | 4679871 | C | T | c.5C>T | A2V | missense | 97% | 2 |
| 20 | 4679877 | T | A | c.11T>A | L4H | missense | 98% | 3 |
| 20 | 4679877 | T | G | c.11T>G | L4R | missense | 98% | 1 |
| 20 | 4679888 | A | G | c.22A>G | M8V | missense | 98% | 1 |
| 20 | 4679901 | T | C | c.35T>C | F12S | missense | 98% | 1 |
| 20 | 4679916 | G | C | c.50G>C | S17T | missense | 98% | 10 |
| 20 | 4679920 | C | A | c.54C>A | D18E | missense | 98% | 2 |
| 20 | 4679920 | C | T | c.54C>T | D18D | synonymous | 98% | 18 |
| 20 | 4679927 | C | A | c.61C>A | L21I | missense | 98% | 1 |
| 20 | 4679932 | C | T | c.66C>T | C22C | synonymous | 98% | 2 |
| 20 | 4679935 | G | A | c.69G>A | K23K | synonymous | 98% | 2 |
| 20 | 4679939 | C | T | c.73C>T | R25C | missense | 98% | 2 |
| 20 | 4679944 | G | A | c.78G>A | P26P | synonymous | 98% | 6 |
| 20 | 4679967 | G | T | c.101G>T | G34V | missense | 98% | 1 |
| 20 | 4679969 | G | A | c.103G>A | G35S | missense | 98% | 1 |
| 20 | 4679975 | C | T | c.109C>T | R37X | nonsense | 98% | 1 |
| 20 | 4679982 | C | T | c.116C>T | P39L | missense | 98% | 3 |
| 20 | 4679983 | G | A | c.117G>A | P39P | synonymous | 98% | 8 |
| 20 | 4679986 | G | A | c.120G>A | G40G | synonymous | 98% | 12 |
| 20 | 4680005 | A | G | c.139A>G | N47D | missense | 98% | 1 |
| 20 | 4680026 | G | A | c.160G>A | G54S | missense | 97% | 78 |
| 20 | 4680028 | T | C | c.162T>C | G54G | synonymous | 97% | 5 |
| 20 | 4680038 | G | T | c.172G>T | G58W | missense | 97% | 1 |
| 20 | 4680045 | C | T | c.179C>T | P60L | missense | 96% | 1 |
| 20 | 4680055 | T | A | c.189T>A | G63G | synonymous | 96% | 1 |
| 20 | 4680077 | G | A | c.211G>A | G71S | missense | 96% | 1 |
| 20 | 4680089 | C | T | c.223C>T | Q75X | nonsense | 96% | 1 |
| 20 | 4680091 | G | A | c.225G>A | Q75Q | synonymous | 96% | 2 |
| 20 | 4680093 | C | G | c.227C>G | P76R | missense | 96% | 1 |
| 20 | 4680129 | G | C | c.263G>C | G88A | missense | 98% | 1 |
| 20 | 4680134 | G | A | c.268G>A | G90S | missense | 98% | 1 |
| 20 | 4680145 | T | G | c.279T>G | G93G | synonymous | 99% | 1 |
| 20 | 4680151 | C | T | c.285C>T | T95T | synonymous | 99% | 1 |
| 20 | 4680172 | G | A | c.306G>A | P102P | synonymous | 99% | 21 |
| 20 | 4680185 | A | G | c.319A>G | T107A | missense | 99% | 1 |
| 20 | 4680199 | C | T | c.333C>T | H111H | synonymous | 99% | 2 |
| 20 | 4680202 | G | A | c.336G>A | M112I | missense | 99% | 1 |
| 20 | 4680231 | T | G | c.365T>G | V122G | missense | 99% | 1 |
| 20 | 4680232 | G | T | c.366G>T | V122V | synonymous | 99% | 3 |
| 20 | 4680244 | C | A | c.378C>A | G126G | synonymous | 99% | 1 |
| 20 | 4680244 | C | T | c.378C>T | G126G | synonymous | 99% | 3 |
| 20 | 4680250 | C | T | c.384C>T | Y128Y | synonymous | 100% | 22 |
| 20 | 4680252 | T | C | c.386T>C | M129T | missense | 100% | 1 |
| 20 | 4680257 | G | T | c.391G>T | G131X | nonsense | 100% | 1 |
| 20 | 4680258 | G | T | c.392G>T | G131V | missense | 100% | 1 |
| 20 | 4680259 | A | G | c.393A>G | G131G | synonymous | 100% | 3 |
| 20 | 4680262 | T | C | c.396T>C | S132S | synonymous | 100% | 1 |
| 20 | 4680274 | G | A | c.408G>A | R136R | synonymous | 100% | 2 |
| 20 | 4680274 | G | T | c.408G>T | R136S | missense | 100% | 2 |
| 20 | 4680279 | T | C | c.413T>C | I138T | missense | 100% | 1 |
| 20 | 4680289 | C | T | c.423C>T | F141F | synonymous | 100% | 2 |
| 20 | 4680292 | C | T | c.426C>T | G142G | synonymous | 100% | 1 |
| 20 | 4680299 | T | G | c.433T>G | Y145D | missense | 100% | 1 |
| 20 | 4680308 | C | T | c.442C>T | R148C | missense | 100% | 1 |
| 20 | 4680309 | G | A | c.443G>A | R148H | missense | 100% | 3 |
| 20 | 4680311 | T | C | c.445T>C | Y149H | missense | 100% | 1 |
| 20 | 4680316 | T | C | c.450T>C | Y150Y | synonymous | 100% | 1 |
| 20 | 4680317 | C | T | c.451C>T | R151C | missense | 100% | 2 |
| 20 | 4680318 | G | A | c.452G>A | R151H | missense | 100% | 3 |
| 20 | 4680324 | A | G | c.458A>G | N153S | missense | 100% | 1 |
| 20 | 4680328 | G | A | c.462G>A | M154I | missense | 100% | 1 |
| 20 | 4680342 | A | G | c.476A>G | N159S | missense | 100% | 1 |
| 20 | 4680349 | G | A | c.483G>A | V161V | synonymous | 100% | 1 |
| 20 | 4680359 | C | T | c.493C>T | P165S | missense | 100% | 2 |
| 20 | 4680362 | A | G | c.496A>G | M166V | missense | 100% | 2 |
| 20 | 4680364 | G | A | c.498G>A | M166I | missense | 100% | 2 |
| 20 | 4680373 | C | T | c.507C>T | Y169Y | synonymous | 100% | 1 |
| 20 | 4680382 | G | A | c.516G>A | Q172Q | synonymous | 100% | 1 |
| 20 | 4680385 | C | T | c.519C>T | N173N | synonymous | 100% | 5 |
| 20 | 4680394 | G | A | c.528G>A | V176V | synonymous | 100% | 2 |
| 20 | 4680397 | C | G | c.531C>G | H177Q | missense | 100% | 1 |
| 20 | 4680397 | C | T | c.531C>T | H177H | synonymous | 100% | 4 |
| 20 | 4680403 | C | T | c.537C>T | C179C | synonymous | 100% | 1 |
| 20 | 4680404 | G | A | c.538G>A | V180I | missense | 100% | 6 |
| 20 | 4680412 | C | G | c.546C>G | I182M | missense | 100% | 2 |
| 20 | 4680429 | C | G | c.563C>G | T188R | missense | 100% | 3 |
| 20 | 4680429 | C | T | c.563C>T | T188M | missense | 100% | 4 |
| 20 | 4680443 | A | G | c.577A>G | T193A | missense | 100% | 2 |
| 20 | 4680445 | C | A | c.579C>A | T193T | synonymous | 100% | 1 |
| 20 | 4680449 | G | C | c.583G>C | G195R | missense | 100% | 3 |
| 20 | 4680451 | G | A | c.585G>A | G195G | synonymous | 100% | 3 |
| 20 | 4680453 | A | C | c.587A>C | E196A | missense | 100% | 9 |
| 20 | 4680462 | C | A | c.596C>A | T199N | missense | 100% | 1 |
| 20 | 4680463 | C | T | c.597C>T | T199T | synonymous | 100% | 2 |
| 20 | 4680467 | A | T | c.601A>T | T201S | missense | 100% | 1 |
| 20 | 4680469 | C | T | c.603C>T | T201T | synonymous | 100% | 3 |
| 20 | 4680470 | G | A | c.604G>A | D202N | missense | 100% | 1 |
| 20 | 4680472 | C | T | c.606C>T | D202D | synonymous | 100% | 8 |
| 20 | 4680473 | G | A | c.607G>A | V203I | missense | 100% | 3 |
| 20 | 4680488 | C | T | c.622C>T | R208C | missense | 100% | 1 |
| 20 | 4680489 | G | A | c.623G>A | R208H | missense | 100% | 9 |
| 20 | 4680490 | C | T | c.624C>T | R208R | synonymous | 100% | 4 |
| 20 | 4680491 | G | A | c.625G>A | V209M | missense | 100% | 1 |
| 20 | 4680494 | G | A | c.628G>A | V210I | missense | 100% | 2 |
| 20 | 4680501 | A | C | c.635A>C | Q212P | missense | 100% | 1 |
| 20 | 4680502 | G | A | c.636G>A | Q212Q | synonymous | 100% | 2 |
| 20 | 4680520 | C | T | c.654C>T | Y218Y | synonymous | 100% | 17 |
| 20 | 4680534 | A | T | c.668A>T | Q223L | missense | 100% | 1 |
| 20 | 4680539 | T | C | c.673T>C | Y225H | missense | 99% | 1 |
| 20 | 4680540 | A | G | c.674A>G | Y225C | missense | 99% | 1 |
| 20 | 4680541 | T | C | c.675T>C | Y225Y | synonymous | 99% | 3 |
| 20 | 4680552 | G | A | c.686G>A | G229E | missense | 98% | 1 |
| 20 | 4680553 | A | G | c.687A>G | G229G | synonymous | 98% | 1 |
| 20 | 4680561 | T | G | c.695T>G | M232R | missense | 97% | 10 |
| 20 | 4680566 | C | T | c.700C>T | L234F | missense | 95% | 29 |
| 20 | 4680590 | C | T | c.724C>T | L242F | missense | 87% | 1 |
| 20 | 4680598 | C | G | c.732C>G | I244M | missense | 84% | 1 |
| 20 | 4680598 | C | T | c.732C>T | I244I | synonymous | 84% | 1 |
| 20 | 4680626 | T | G | c.760T>G | X254G | read-through | 66% | 1 |

#### <a name="table_s4">Table S4</a>. Summary of rare *PRNP* variants by functional class in ExAC

| Class | Total AC |
| ---- | ---- |
| missense | 236 |
| non-coding | 1 |
| nonsense | 3 |
| read-through | 1 |
| synonymous | 180 |

#### <a name="table_s5">Table S5</a>. Allele counts of 16 reportedly pathogenic *PRNP* variants in >500,000 23andMe research participants.

To protect the privacy of 23andMe research participants, allele count (AC) values between 1 and 5 inclusive are displayed as "1-5“ and are rounded up to 5 for the purposes of plotting. These alleles were seen almost exclusively in a heterozygous state, with fewer than 5 homozygous individuals total across all 16 variants.

<table>
  <tr>
    <td>Variant</td>
    <td>dbSNP id</td>
    <td>23andMe id</td>
    <td>Called genotypes</td>
    <td>AC</td>
    <td>Comments</td>
  </tr>
  <tr>
    <td>P102L</td>
    <td>rs74315401</td>
    <td>i5004359</td>
    <td>502075</td>
    <td rowspan="4">1-5 total</td>
    <td></td>
  </tr>
  <tr>
    <td>A117V</td>
    <td>rs74315402</td>
    <td>i5004358</td>
    <td>501820</td>
    <td></td>
  </tr>
  <tr>
    <td>D178N</td>
    <td>rs74315403</td>
    <td>i5004357</td>
    <td>502450</td>
    <td></td>
  </tr>
  <tr>
    <td>E200K</td>
    <td>rs28933385</td>
    <td>rs28933385</td>
    <td>531370</td>
    <td></td>
  </tr>
  <tr>
    <td>M232R</td>
    <td>rs74315409</td>
    <td>i5004352</td>
    <td>502475</td>
    <td>78</td>
    <td>AC=29 in 2,685 individuals with >90% Japanese ancestry</td>
  </tr>
  <tr>
    <td>V180I</td>
    <td>rs74315408</td>
    <td>i5004353</td>
    <td>502125</td>
    <td>15</td>
    <td>AC=1-5 in 2,670 individuals with >90% Japanese ancestry</td>
  </tr>
  <tr>
    <td>V210I</td>
    <td>rs74315407</td>
    <td>i5004354</td>
    <td>502290</td>
    <td>13</td>
    <td>AC=8 in 385,030 Europeans</td>
  </tr>
  <tr>
    <td>R208C</td>
    <td>rs55826236</td>
    <td>rs55826236</td>
    <td>501850</td>
    <td>8</td>
    <td></td>
  </tr>
  <tr>
    <td>R208H</td>
    <td>rs74315412</td>
    <td>i5004349</td>
    <td>501775</td>
    <td>22</td>
    <td>AC=19 in 384,645 Europeans</td>
  </tr>
  <tr>
    <td>P105L</td>
    <td>rs11538758</td>
    <td>rs11538758</td>
    <td>531575</td>
    <td rowspan="7">1-5 total</td>
    <td></td>
  </tr>
  <tr>
    <td>G131V</td>
    <td>rs74315410</td>
    <td>i5004351</td>
    <td>499455</td>
    <td></td>
  </tr>
  <tr>
    <td>A133V</td>
    <td>rs74315415</td>
    <td>i5004347</td>
    <td>502520</td>
    <td></td>
  </tr>
  <tr>
    <td>T183A</td>
    <td>rs74315411</td>
    <td>i5004350</td>
    <td>502295</td>
    <td></td>
  </tr>
  <tr>
    <td>F198V</td>
    <td>rs55871421</td>
    <td>rs55871421</td>
    <td>501540</td>
    <td></td>
  </tr>
  <tr>
    <td>F198S</td>
    <td>rs74315405</td>
    <td>i5004356</td>
    <td>502460</td>
    <td></td>
  </tr>
  <tr>
    <td>G217R</td>
    <td>rs74315406</td>
    <td>i5004355</td>
    <td>502385</td>
    <td></td>
  </tr>
</table>

#### <a name="table_s6">Table S6</a>. Phenotypes investigated in studies in which ExAC individuals with reportedly pathogenic PRNP variants were ascertained. 

Note that we do not have access to phenotypic data to indicate whether a particular individual was ascertained as a case or a control. Therefore "cardiovascular" simply means an individual was ascertained in a cardiovascular disease cohort, not necessarily that the individual has cardiovascular disease. “Mixed” cohorts include controls, cardiovascular and pulmonary phenotypes.

| Cohort phenotype | Total in ExAC | With reportedly pathogenic *PRNP* variants |
| ---- | ---- | ---- |
| Autoimmune | 1675 | 4 |
| Cancer | 7601 | 3 |
| Cardiovascular | 14622 | 14 |
| Metabolic | 15327 | 19 |
| Mixed | 3936 | 2 |
| Population controls | 2215 | 6 |
| Psychiatric | 15330 | 4 |
| TOTAL | 60706 | 52 |

#### <a name="table_s7">Table S7</a>. Inferred ancestry and codon 129 genotypes of ExAC individuals with reportedly pathogenic variants.

Three-letter HapMap ancestry codes are defined in [Table S8](#table_s8).

| variant | pops | codon129 | 
| ---- | ---- | ---- | 
| P39L |  1 PJL, 2 TSI |  2 M/M, 1 M/V | 
| G131V |  1 TSI |  1 M/V | 
| R148H |  1 CEU, 1 IBS, 1 PJL |  3 M/M | 
| V180I |  1 CHB, 2 JPT, 3 PJL |  4 M/M, 1 M/V, 1 V/V |
| T188R |  1 CLM, 2 MXL |  1 M/V, 2 V/V | 
| E196A |  3 CHB, 6 CHS |  9 M/M |  
| D202N |  1 TSI |  1 M/V | 
| V203I |  1 IBS, 2 TSI |  1 M/M, 2 M/V | 
| R208C |  1 ACB |  1 M/M | 
| R208H |  1 ACB, 2 ASW, 1 CLM, 2 IBS, 1 MSL, 2 TSI |  4 M/M, 5 M/V | 
| V210I |  2 TSI |  2 M/M | 
| Q212P |  1 CEU |  1 M/V | 
| M232R |  5 CHB, 5 JPT |  10 M/M | 

#### <a name="table_s8">Table S8</a>. Inferred ancestry of all ExAC individuals.

Methods for ancestry assignment are described in Materials and Methods.

| Population code | Description | Super population code | N in ExAC |
| ---- | ---- | ---- | ---- |
| ACB | African Caribbeans in Barbados | AFR | 2267 |
| ASW | Americans of African Ancestry in SW USA | AFR | 2151 |
| BEB | Bengali from Bangladesh | SAS | 483 |
| CDX | Chinese Dai in Xishuangbanna, China | EAS | 19 |
| CEU | Utah Residents (CEPH) with Northern and Western European ancestry | EUR | 14185 |
| CHB | Han Chinese in Beijing, China | EAS | 1553 |
| CHS | Southern Han Chinese | EAS | 1733 |
| CLM | Colombians from Medellin, Colombia | AMR | 870 |
| ESN | Esan in Nigeria | AFR | 89 |
| FIN | Finnish in Finland | EUR | 3977 |
| GBR | British in England and Scotland | EUR | 10358 |
| GIH | Gujarati Indian from Houston, Texas | SAS | 79 |
| GWD | Gambian in Western Divisions in The Gambia | AFR | 102 |
| IBS | Iberian population in Spain | EUR | 3534 |
| ITU | Indian Telugu from the UK | SAS | 1089 |
| JPT | Japanese in Tokyo, Japan | EAS | 663 |
| KHV | Kinh in Ho Chi Minh City, Vietnam | EAS | 369 |
| LWK | Luhya in Webuye, Kenya | AFR | 72 |
| MSL | Mende in Sierra Leone | AFR | 189 |
| MXL | Mexican Ancestry from Los Angeles USA | AMR | 2658 |
| PEL | Peruvians from Lima, Peru | AMR | 1900 |
| PJL | Punjabi from Lahore, Pakistan | SAS | 6300 |
| PUR | Puerto Ricans from Puerto Rico | AMR | 579 |
| STU | Sri Lankan Tamil from the UK | SAS | 460 |
| TSI | Toscani in Italia | EUR | 4795 |
| YRI | Yoruba in Ibadan, Nigeria | AFR | 232 |

#### <a name="table_s9">Table S9</a>. Inferred ancestry of 23andMe research participants

| Ancestry | Minimum called genotypes | Maximum called genotypes | Total reportedly pathogenic AC |
| ---- | ---- | ---- | ---- |
| European | 382865 | 408475 | 35 |
| Latino | 42425 | 44480 | 10 |
| African | 22945 | 23795 | 10 |
| East Asian | 20255 | 21710 | 75 |
| All others | 30975 | 33125 | 20 |
| TOTAL | 499455 | 531575 | 140 |

#### <a name="table_s10">Table S10</a>. Details of Japanese prion disease cases**

+ Age at onset is expressed as the mean ± SD (range) years.
+ Duration between the onset and akinetic mutism or death without akinetic mutism. Duration is expressed as the mean ± SD (range) months.
+ Terms:
    + EE = glutamic acid homozygosity
    + EK = glutamic acid/lysine heterozygosity
    + KK = lysine homozygosity
    + MM = methionine homozygosity
    + MV = methionine/valine heterozygosity
    + PSWCs = periodic synchronous wave complexes

<table>
  <tr>
    <td>Variant</td>
    <td>N</td>
    <td>Male/Female</td>
    <td>Age at onset*</td>
    <td>(range)</td>
    <td>Positive family history (%)</td>
  </tr>
  <tr>
    <td>Insertion</td>
    <td>8</td>
    <td>4/4</td>
    <td>51.0 ± 12.0</td>
    <td>(26-68)</td>
    <td>5 (63)</td>
  </tr>
  <tr>
    <td>P102L</td>
    <td>83</td>
    <td>38/45</td>
    <td>55.5 ± 10.3</td>
    <td>(22-75)</td>
    <td>69 (83)</td>
  </tr>
  <tr>
    <td>P105L</td>
    <td>12</td>
    <td>7/5</td>
    <td>46.9 ± 8.4</td>
    <td>(31-61)</td>
    <td>11 (92)</td>
  </tr>
  <tr>
    <td>D178N-129M</td>
    <td>4</td>
    <td>3/1</td>
    <td>54.5 ± 5.5</td>
    <td>(46-61)</td>
    <td>None</td>
  </tr>
  <tr>
    <td>D178N-129V</td>
    <td>1</td>
    <td>1/0</td>
    <td>74</td>
    <td></td>
    <td>None</td>
  </tr>
  <tr>
    <td>V180I</td>
    <td>218</td>
    <td>84/134</td>
    <td>77.4 ± 6.8</td>
    <td>(44-93)</td>
    <td>5 (2)</td>
  </tr>
  <tr>
    <td>E200K</td>
    <td>63</td>
    <td>30/33</td>
    <td>61.1 ± 9.9</td>
    <td>(31-83)</td>
    <td>28 (44)</td>
  </tr>
  <tr>
    <td>V203I</td>
    <td>3</td>
    <td>2/1</td>
    <td>73</td>
    <td></td>
    <td>None</td>
  </tr>
  <tr>
    <td>R208H</td>
    <td>1</td>
    <td>0/1</td>
    <td>74</td>
    <td></td>
    <td>None</td>
  </tr>
  <tr>
    <td>V210I</td>
    <td>1</td>
    <td>0/1</td>
    <td>55</td>
    <td></td>
    <td>None</td>
  </tr>
  <tr>
    <td>M232R</td>
    <td>63</td>
    <td>32/31</td>
    <td>64.4 ± 10.9</td>
    <td>(15-82)</td>
    <td>2 (3)</td>
  </tr>
  <tr>
    <td>V180I+M232R</td>
    <td>4</td>
    <td>2/2</td>
    <td>71.3 ± 3.6</td>
    <td>(65-74)</td>
    <td>None</td>
  </tr>
</table>

<table>
  <tr>
    <td>Variant</td>
    <td>Duration**</td>
    <td>(range)</td>
    <td>Codon 129</td>
    <td>Codon 219</td>
  </tr>
  <tr>
    <td>Insertion</td>
    <td>27.8 ± 17.7</td>
    <td>(3-57)</td>
    <td>MM 6; MV 1</td>
    <td>EE 6; KK 1</td>
  </tr>
  <tr>
    <td>P102L</td>
    <td>48.4 ± 35.8</td>
    <td>(2-186)</td>
    <td>MM 67; MV 6</td>
    <td>EE 70; EK 2</td>
  </tr>
  <tr>
    <td>P105L</td>
    <td>90.2 ± 40.4</td>
    <td>(25-184)</td>
    <td>MV 11</td>
    <td>EE 7</td>
  </tr>
  <tr>
    <td>D178N-129M</td>
    <td>8.5 ± 4.4</td>
    <td>(2-13)</td>
    <td>MM 4</td>
    <td>EE 4</td>
  </tr>
  <tr>
    <td>D178N-129V</td>
    <td>24</td>
    <td></td>
    <td>MV 1</td>
    <td>EE 1</td>
  </tr>
  <tr>
    <td>V180I</td>
    <td>16.4 ± 14.5</td>
    <td>(0-70)</td>
    <td>MM 162; MV 54</td>
    <td>EE 210</td>
  </tr>
  <tr>
    <td>E200K</td>
    <td>5.0 ± 6.0</td>
    <td>(1-32)</td>
    <td>MM 58; MV 3</td>
    <td>EE 58; EK 3</td>
  </tr>
  <tr>
    <td>V203I</td>
    <td>3.7 ± 2.1</td>
    <td>(1-6)</td>
    <td>MM 3</td>
    <td>EE 3</td>
  </tr>
  <tr>
    <td>R208H</td>
    <td>3</td>
    <td></td>
    <td>MM 1</td>
    <td>EE 1</td>
  </tr>
  <tr>
    <td>V210I</td>
    <td>3</td>
    <td></td>
    <td>MM 1</td>
    <td>EE 1</td>
  </tr>
  <tr>
    <td>M232R</td>
    <td>8.6 ± 12.7</td>
    <td>(0-78)</td>
    <td>MM 60; MV 2</td>
    <td>EE 61; EK 1</td>
  </tr>
  <tr>
    <td>V180I+M232R</td>
    <td>21.8 ± 17.7</td>
    <td>(1-47)</td>
    <td>MM 4</td>
    <td>EE 4</td>
  </tr>
</table>


<table>
  <tr>
    <td>Variant</td>
    <td>PSWCs on EEG (%)</td>
    <td>Hyperintensities on MRI (%)</td>
    <td>Positive 14-3-3 protein (%)</td>
  </tr>
  <tr>
    <td>Insertion</td>
    <td>3/8 (38)</td>
    <td>2/7 (29)</td>
    <td>0/1 (0)</td>
  </tr>
  <tr>
    <td>P102L</td>
    <td>11/72 (15)</td>
    <td>32/76 (42)</td>
    <td>13/34 (38)</td>
  </tr>
  <tr>
    <td>P105L</td>
    <td>1/10 (10)</td>
    <td>1/11 (9)</td>
    <td>1/2 (50)</td>
  </tr>
  <tr>
    <td>D178N-129M</td>
    <td>0/4 (0)</td>
    <td>1/4 (25)</td>
    <td>1/2 (50)</td>
  </tr>
  <tr>
    <td>D178N-129V</td>
    <td>0/1 (0)</td>
    <td>0/1 (0)</td>
    <td>1/1 (100)</td>
  </tr>
  <tr>
    <td>V180I</td>
    <td>19/203 (9)</td>
    <td>212/213 (99)</td>
    <td>110/140 (79)</td>
  </tr>
  <tr>
    <td>E200K</td>
    <td>56/63 (89)</td>
    <td>56/59 (95)</td>
    <td>29/31 (94)</td>
  </tr>
  <tr>
    <td>V203I</td>
    <td>3/3 (100)</td>
    <td>2/2 (100)</td>
    <td>1/1 (100)</td>
  </tr>
  <tr>
    <td>R208H</td>
    <td>1/1 (100)</td>
    <td>1/1 (100)</td>
    <td>1/1 (100)</td>
  </tr>
  <tr>
    <td>V210I</td>
    <td>1/1 (100)</td>
    <td>1/1 (100)</td>
    <td>not done</td>
  </tr>
  <tr>
    <td>M232R</td>
    <td>46/61 (75)</td>
    <td>55/60 (92)</td>
    <td>31/43 (72)</td>
  </tr>
  <tr>
    <td>V180I+M232R</td>
    <td>0/4 (0)</td>
    <td>4/4 (100)</td>
    <td>0/1 (0)</td>
  </tr>
</table>

#### <a name="table_s11">Table S11</a>. Phenotypes of individuals with N-terminal PrP truncating variants

| HGVS | Variant | Zygosity | Sex | Age | Available phenotype information |
| ---- | ---- | ---- | ---- | ---- | ---- |
| c.59_60insC | G20Gfs84X | Het | F | 79 | Ascertained as part of the Rotterdam Study [[Hofman 2015]], a prospective cohort study of middle-aged and elderly persons. In good health and free of dementia as of at least age 78, at last in-person examination completion. Has 5 siblings and 2 children. Only family history noted is that one sibling has had a stroke before age 65. |
| c.109C>T | R37X | Het | M | 73 | Ascertained as a control for the Swedish schizophrenia study. Underwent heart bypass surgery in 2008, has a family history of heart problems. 4 siblings. Reports no family history of neurodegeneration or neuropathy. |
| c.223C>T | Q75X | Het | M | 52 | Ascertained in a study of type 2 diabetes. Has mild type 2 diabetes treated with metformin. Has children. |
| c.391G>T | G131X | Het | F |  | None available. |

#### <a name="figure_s1">Figure S1</a>. Age of ExAC individuals with reportedly pathogenic *PRNP* variants versus all individuals in ExAC.

The distribution of ages, available for 40 of 52 individuals with reportedly pathogenic *PRNP* variants, did not differ from the distribution overall (p = .69, Wilcoxon rank-sum test; p = .69, student's t test) nor after controlling for cohort (p = .15, linear regression).

![Figure S1](/figures/raster/figure_s1.png)

#### <a name="figure_s2">Figure S2</a>. Sanger sequencing results for individuals with N-terminal truncating variants

**Figure S2A.** G20Gfs84X reverse (top) and forward (bottom). Primers: 2a-forward: AACTTAGGGTCACATTTGTCCTTGG; 2a-reverse: GGTAACGGTGCATGTTTTCACG. 2b forward: GTGGTGGCTGGGGTCAAGG; 2b reverse: TTTCCAGTGCCCATCAGTGC.

![Figure S2A](/figures/raster/figure_s2a.png)

**Figure S2B.** R37X - DNA from whole blood (top) and fibroblasts (bottom). Primers: PrP2-F: TGGGACTCTGACGTTCTCCT; PrP2-R: GGTGAAGTTCTCCCCCTTGG

![Figure S2B](/figures/raster/figure_s2b.png)

**Figure S2C.** Q75X. Primers: PRNP_EX2-M13-F  \[TGTAAAACGACGGCCAGT\] CCATTGCTATGCACTCATTCA;  PRNP_EX2-M13-R  \[CAGGAAACAGCTATGACC\] CCATGTGCTTCATGTTGGTT

![Figure S2C](/figures/raster/figure_s2c.png)


[Aguzzi 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18558863 "Aguzzi A, Baumann F, Bremer J. The prion's elusive reason for being. Annu Rev Neurosci. 2008;31:439-77. doi: 10.1146/annurev.neuro.31.060407.125620. Review.PubMed PMID: 18558863."
[Ahn 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24866748 "Ahn M, Bajsarowicz K, Oehler A, Lemus A, Bankiewicz K, DeArmond SJ.Convection-enhanced delivery of AAV2-PrPshRNA in prion-infected mice. PLoS One.2014 May 27;9(5):e98496. doi: 10.1371/journal.pone.0098496. eCollection 2014.PubMed PMID: 24866748; PubMed Central PMCID: PMC4035323."
[Ahn 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24866748 "Ahn M, Bajsarowicz K, Oehler A, Lemus A, Bankiewicz K, DeArmond SJ.Convection-enhanced delivery of AAV2-PrPshRNA in prion-infected mice. PLoS One.2014 May 27;9(5):e98496. doi: 10.1371/journal.pone.0098496. eCollection 2014.PubMed PMID: 24866748; PubMed Central PMCID: PMC4035323."
[Alzualde 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20613639 "Alzualde A, Indakoetxea B, Ferrer I, Moreno F, Barandiaran M, Gorostidi A,Estanga A, Ruiz I, Calero M, van Leeuwen FW, Atares B, Juste R,Rodriguez-Martínez AB, López de Munain A. A novel PRNP Y218N mutation inGerstmann-Sträussler-Scheinker disease with neurofibrillary degeneration. JNeuropathol Exp Neurol. 2010 Aug;69(8):789-800. doi:10.1097/NEN.0b013e3181e85737. PubMed PMID: 20613639."
[Amberger 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21472891 "Amberger J, Bocchini C, Hamosh A. A new face and new challenges for OnlineMendelian Inheritance in Man (OMIM®). Hum Mutat. 2011 May;32(5):564-7. doi:10.1002/humu.21466. Epub 2011 Apr 5. PubMed PMID: 21472891."
[Basset-Leobon 2006]: http://www.ncbi.nlm.nih.gov/pubmed/16533975 "Basset-Leobon C, Uro-Coste E, Peoc'h K, Haik S, Sazdovitch V, Rigal M,Andreoletti O, Hauw JJ, Delisle MB. Familial Creutzfeldt-Jakob disease with anR208H-129V haplotype and Kuru plaques. Arch Neurol. 2006 Mar;63(3):449-52. PubMedPMID: 16533975."
[Beck 2001]: http://www.ncbi.nlm.nih.gov/pubmed/11468331 "Beck JA, Mead S, Campbell TA, Dickinson A, Wientjens DP, Croes EA, Van DuijnCM, Collinge J. Two-octapeptide repeat deletion of prion protein associated with rapidly progressive dementia. Neurology. 2001 Jul 24;57(2):354-6. PubMed PMID:11468331."
[Beck 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20583301 "Beck JA, Poulter M, Campbell TA, Adamson G, Uphill JB, Guerreiro R, JacksonGS, Stevens JC, Manji H, Collinge J, Mead S. PRNP allelic series from 19 years ofprion protein gene sequencing at the MRC Prion Unit. Hum Mutat. 2010Jul;31(7):E1551-63. doi: 10.1002/humu.21281. PubMed PMID: 20583301."
[Beck 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22108575 "Beck J, Collinge J, Mead S. Prion protein gene M232R variation is probably an uncommon polymorphism rather than a pathogenic mutation. Brain. 2012 Feb;135(Pt2):e209; author reply e210. doi: 10.1093/brain/awr294. Epub 2011 Nov 21. PubMedPMID: 22108575."
[Begg 2002]: http://www.ncbi.nlm.nih.gov/pubmed/12189225 "Begg CB. On the use of familial aggregation in population-based case probands for calculating penetrance. J Natl Cancer Inst. 2002 Aug 21;94(16):1221-6. PubMedPMID: 12189225."
[Benestad 2012]: http://www.ncbi.nlm.nih.gov/pubmed/23249298 "Benestad SL, Austbø L, Tranulis MA, Espenes A, Olsaker I. Healthy goatsnaturally devoid of prion protein. Vet Res. 2012 Dec 18;43:87. doi:10.1186/1297-9716-43-87. PubMed PMID: 23249298; PubMed Central PMCID: PMC3542104."
[Bernardi 2014]: http://www.ncbi.nlm.nih.gov/pubmed/25022973 "Bernardi L, Cupidi C, Frangipane F, Anfossi M, Gallo M, Conidi ME, Vasso F,Colao R, Puccio G, Curcio SA, Mirabelli M, Clodomiro A, Di Lorenzo R, Smirne N,Maletta R, Bruni AC. Novel N-terminal domain mutation in prion protein detectedin 2 patients diagnosed with frontotemporal lobar degeneration syndrome.Neurobiol Aging. 2014 Nov;35(11):2657.e7-11. doi:10.1016/j.neurobiolaging.2014.06.006. Epub 2014 Jun 14. PubMed PMID: 25022973."
[Bick 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22958901 "Bick AG, Flannick J, Ito K, Cheng S, Vasan RS, Parfenov MG, Herman DS, DePalmaSR, Gupta N, Gabriel SB, Funke BH, Rehm HL, Benjamin EJ, Aragam J, Taylor HA Jr, Fox ER, Newton-Cheh C, Kathiresan S, O'Donnell CJ, Wilson JG, Altshuler DM,Hirschhorn JN, Seidman JG, Seidman C. Burden of rare sarcomere gene variants inthe Framingham and Jackson Heart Study cohorts. Am J Hum Genet. 2012 Sep7;91(3):513-9. doi: 10.1016/j.ajhg.2012.07.017. PubMed PMID: 22958901; PubMedCentral PMCID: PMC3511985."
[Bishop 2009]: http://www.ncbi.nlm.nih.gov/pubmed/20035629 "Bishop MT, Pennington C, Heath CA, Will RG, Knight RS. PRNP variation in UKsporadic and variant Creutzfeldt Jakob disease highlights genetic risk factorsand a novel non-synonymous polymorphism. BMC Med Genet. 2009 Dec 26;10:146. doi: 10.1186/1471-2350-10-146. PubMed PMID: 20035629; PubMed Central PMCID:PMC2806268."
[Bouybayoune 2015]: http://www.ncbi.nlm.nih.gov/pubmed/25880443 "Bouybayoune I, Mantovani S, Del Gallo F, Bertani I, Restelli E, Comerio L,Tapella L, Baracchi F, Fernández-Borges N, Mangieri M, Bisighini C, Beznoussenko GV, Paladini A, Balducci C, Micotti E, Forloni G, Castilla J, Fiordaliso F,Tagliavini F, Imeri L, Chiesa R. Transgenic fatal familial insomnia mice indicateprion infectivity-independent mechanisms of pathogenesis and phenotypicexpression of disease. PLoS Pathog. 2015 Apr 16;11(4):e1004796. doi:10.1371/journal.ppat.1004796. eCollection 2015 Apr. Erratum in: PLoS Pathog. 2015Jul;11(7):e1005046. PubMed PMID: 25880443; PubMed Central PMCID: PMC4400166."
[Brandel 2000]: http://www.ncbi.nlm.nih.gov/pubmed/22057088 "Brandel JP, Welaratne A, Salomon D, Capek I, Vaillant V, Aouba A, Haïk S,Alpérovitch A. Can mortality data provide reliable indicators forCreutzfeldt-Jakob disease surveillance? A study in France from 2000 to 2008.Neuroepidemiology. 2011;37(3-4):188-92. doi: 10.1159/000332764. Epub 2011 Nov 2. Erratum in: Neuroepidemiology. 2012;38(3):193. Aouaba, Albertine [corrected toAouba, Albertine]. PubMed PMID: 22057088."
[Brandel 2011]: http://www.ncbi.nlm.nih.gov/pubmed/22057088 "Brandel JP, Welaratne A, Salomon D, Capek I, Vaillant V, Aouba A, Haïk S, Alpérovitch A. Can mortality data provide reliable indicators for Creutzfeldt-Jakob disease surveillance? A study in France from 2000 to 2008. Neuroepidemiology. 2011;37(3-4):188-92. doi: 10.1159/000332764. Epub 2011 Nov 2.  Erratum in: Neuroepidemiology. 2012;38(3):193. Aouaba, Albertine [corrected to Aouba, Albertine]. PubMed PMID: 22057088."
[Bratosiewicz 2000]: http://www.ncbi.nlm.nih.gov/pubmed/11693719 "Bratosiewicz J, Barcikowska M, Cervenakowa L, Brown P, Gajdusek DC, LiberskiPP. A new point mutation of the PRNP gene in Gerstmann-Sträussler-Scheinker case in Poland. Folia Neuropathol. 2000;38(4):164-6. PubMed PMID: 11693719."
[Bremer 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20098419 "Bremer J, Baumann F, Tiberi C, Wessig C, Fischer H, Schwarz P, Steele AD,Toyka KV, Nave KA, Weis J, Aguzzi A. Axonal prion protein is required forperipheral myelin maintenance. Nat Neurosci. 2010 Mar;13(3):310-8. doi:10.1038/nn.2483. Epub 2010 Jan 24. PubMed PMID: 20098419."
[Browning & Browning 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17924348 "Browning SR, Browning BL. Rapid and accurate haplotype phasing andmissing-data inference for whole-genome association studies by use of localizedhaplotype clustering. Am J Hum Genet. 2007 Nov;81(5):1084-97. Epub 2007 Sep 21.PubMed PMID: 17924348; PubMed Central PMCID: PMC2265661."
[Brunham & Hayden 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23504071 "Brunham LR, Hayden MR. Hunting human disease genes: lessons from the past,challenges for the future. Hum Genet. 2013 Jun;132(6):603-17. doi:10.1007/s00439-013-1286-3. Epub 2013 Mar 17. Review. PubMed PMID: 23504071;PubMed Central PMCID: PMC3654184."
[Bryc 2015]: http://www.ncbi.nlm.nih.gov/pubmed/25529636 "Bryc K, Durand EY, Macpherson JM, Reich D, Mountain JL. The genetic ancestryof African Americans, Latinos, and European Americans across the United States.Am J Hum Genet. 2015 Jan 8;96(1):37-53. doi: 10.1016/j.ajhg.2014.11.010. Epub2014 Dec 18. PubMed PMID: 25529636; PubMed Central PMCID: PMC4289685."
[Bueler 1992]: http://www.ncbi.nlm.nih.gov/pubmed/1373228 "Büeler H, Fischer M, Lang Y, Bluethmann H, Lipp HP, DeArmond SJ, Prusiner SB, Aguet M, Weissmann C. Normal development and behaviour of mice lacking theneuronal cell-surface PrP protein. Nature. 1992 Apr 16;356(6370):577-82. PubMedPMID: 1373228."
[Bueler 1993]: http://www.ncbi.nlm.nih.gov/pubmed/8100741 "Büeler H, Aguzzi A, Sailer A, Greiner RA, Autenried P, Aguet M, Weissmann C.Mice devoid of PrP are resistant to scrapie. Cell. 1993 Jul 2;73(7):1339-47.PubMed PMID: 8100741."
[Butefisch 2000]: http://www.ncbi.nlm.nih.gov/pubmed/10953183 "Bütefisch CM, Gambetti P, Cervenakova L, Park KY, Hallett M, Goldfarb LG.Inherited prion encephalopathy associated with the novel PRNP H187R mutation: aclinical study. Neurology. 2000 Aug 22;55(4):517-22. PubMed PMID: 10953183."
[Campbell 1996]: http://www.ncbi.nlm.nih.gov/pubmed/8618679 "Campbell TA, Palmer MS, Will RG, Gibb WR, Luthert PJ, Collinge J. A priondisease with a novel 96-base pair insertional mutation in the prion protein gene.Neurology. 1996 Mar;46(3):761-6. PubMed PMID: 8618679."
[Capellari 2002]: http://www.ncbi.nlm.nih.gov/pubmed/12451210 "Capellari S, Parchi P, Wolff BD, Campbell J, Atkinson R, Posey DM, PetersenRB, Gambetti P. Creutzfeldt-Jakob disease associated with a deletion of tworepeats in the prion protein gene. Neurology. 2002 Nov 26;59(10):1628-30. PubMed PMID: 12451210."
[Capellari 2005]: http://www.ncbi.nlm.nih.gov/pubmed/15753435 "Capellari S, Cardone F, Notari S, Schininà ME, Maras B, Sità D, Baruzzi A,Pocchiari M, Parchi P. Creutzfeldt-Jakob disease associated with the R208Hmutation in the prion protein gene. Neurology. 2005 Mar 8;64(5):905-7. PubMedPMID: 15753435."
[Capellari 2011]: http://www.ncbi.nlm.nih.gov/pubmed/20978903 "Capellari S, Strammiello R, Saverioni D, Kretzschmar H, Parchi P. GeneticCreutzfeldt-Jakob disease and fatal familial insomnia: insights into phenotypicvariability and disease pathogenesis. Acta Neuropathol. 2011 Jan;121(1):21-37.doi: 10.1007/s00401-010-0760-4. Epub 2010 Oct 27. Review. PubMed PMID: 20978903."
[Chapman 1994]: http://www.ncbi.nlm.nih.gov/pubmed/7936296 "Chapman J, Ben-Israel J, Goldhammer Y, Korczyn AD. The risk of developingCreutzfeldt-Jakob disease in subjects with the PRNP gene codon 200 pointmutation. Neurology. 1994 Sep;44(9):1683-6. PubMed PMID: 7936296."
[Chasseigneaux 2006]: http://www.ncbi.nlm.nih.gov/pubmed/17029785 "Chasseigneaux S, Haïk S, Laffont-Proust I, De Marco O, Lenne M, Brandel JP,Hauw JJ, Laplanche JL, Peoc'h K. V180I mutation of the prion protein geneassociated with atypical PrPSc glycosylation. Neurosci Lett. 2006 Nov20;408(3):165-9. Epub 2006 Oct 9. PubMed PMID: 17029785."
[Chen 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21791975 "Chen C, Shi Q, Tian C, Li Q, Zhou W, Gao C, Han J, Dong XP. The first Chinese case of Creutzfeldt-Jakob disease patient with R208H mutation in PRNP. Prion.2011 Jul-Sep;5(3):232-4. doi: 10.4161/pri.5.3.16796. Epub 2011 Jul 1. PubMedPMID: 21791975; PubMed Central PMCID: PMC3226051."
[Chesebro 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20221436 "Chesebro B, Race B, Meade-White K, Lacasse R, Race R, Klingeborn M, StriebelJ, Dorward D, McGovern G, Jeffrey M. Fatal transmissible amyloid encephalopathy: a new type of prion disease associated with lack of prion protein membraneanchoring. PLoS Pathog. 2010 Mar 5;6(3):e1000800. doi:10.1371/journal.ppat.1000800. PubMed PMID: 20221436; PubMed Central PMCID:PMC2832701."
[Chong 2015]: http://www.ncbi.nlm.nih.gov/pubmed/26166479 "Chong JX, Buckingham KJ, Jhangiani SN, Boehm C, Sobreira N, Smith JD, Harrell TM, McMillin MJ, Wiszniewski W, Gambin T, Coban Akdemir ZH, Doheny K, Scott AF,Avramopoulos D, Chakravarti A, Hoover-Fong J, Mathews D, Witmer PD, Ling H,Hetrick K, Watkins L, Patterson KE, Reinier F, Blue E, Muzny D, Kircher M,Bilguvar K, López-Giráldez F, Sutton VR, Tabor HK, Leal SM, Gunel M, Mane S,Gibbs RA, Boerwinkle E, Hamosh A, Shendure J, Lupski JR, Lifton RP, Valle D,Nickerson DA; Centers for Mendelian Genomics, Bamshad MJ. The Genetic Basis ofMendelian Phenotypes: Discoveries, Challenges, and Opportunities. Am J Hum Genet.2015 Aug 6;97(2):199-215. doi: 10.1016/j.ajhg.2015.06.009. Epub 2015 Jul 9.Review. PubMed PMID: 26166479; PubMed Central PMCID: PMC4573249."
[Collins 1970]: http://www.ncbi.nlm.nih.gov/pubmed/12427885 "Collins S, Boyd A, Lee JS, Lewis V, Fletcher A, McLean CA, Law M, Kaldor J,Smith MJ, Masters CL. Creutzfeldt-Jakob disease in Australia 1970-1999.Neurology. 2002 Nov 12;59(9):1365-71. PubMed PMID: 12427885."
[Collins 2000]: http://www.ncbi.nlm.nih.gov/pubmed/10891990 "Collins S, Boyd A, Fletcher A, Byron K, Harper C, McLean CA, Masters CL. Novelprion protein gene mutation in an octogenarian with Creutzfeldt-Jakob disease.Arch Neurol. 2000 Jul;57(7):1058-63. PubMed PMID: 10891990."
[Collins 2002]: http://www.ncbi.nlm.nih.gov/pubmed/12427885 "Collins S, Boyd A, Lee JS, Lewis V, Fletcher A, McLean CA, Law M, Kaldor J, Smith MJ, Masters CL. Creutzfeldt-Jakob disease in Australia 1970-1999. Neurology. 2002 Nov 12;59(9):1365-71. PubMed PMID: 12427885."
[Cooper 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21841781 "Cooper GM, Coe BP, Girirajan S, Rosenfeld JA, Vu TH, Baker C, Williams C,Stalker H, Hamid R, Hannig V, Abdel-Hamid H, Bader P, McCracken E, Niyazov D,Leppig K, Thiese H, Hummel M, Alexander N, Gorski J, Kussmann J, Shashi V,Johnson K, Rehder C, Ballif BC, Shaffer LG, Eichler EE. A copy number variationmorbidity map of developmental delay. Nat Genet. 2011 Aug 14;43(9):838-46. doi:10.1038/ng.909. Erratum in: Nat Genet. 2014 Sep;46(9):1040. PubMed PMID:21841781; PubMed Central PMCID: PMC3171215."
[Cooper 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23820649 "Cooper DN, Krawczak M, Polychronakos C, Tyler-Smith C, Kehrer-Sawatzki H.Where genotype is not predictive of phenotype: towards an understanding of themolecular basis of reduced penetrance in human inherited disease. Hum Genet. 2013Oct;132(10):1077-130. doi: 10.1007/s00439-013-1331-2. Epub 2013 Jul 3. Review.PubMed PMID: 23820649; PubMed Central PMCID: PMC3778950."
[Cornfield 1951]: http://www.ncbi.nlm.nih.gov/pubmed/14861651 "CORNFIELD J. A method of estimating comparative rates from clinical data;applications to cancer of the lung, breast, and cervix. J Natl Cancer Inst. 1951 Jun;11(6):1269-75. PubMed PMID: 14861651."
[Crow 1999]: http://www.ncbi.nlm.nih.gov/pubmed/10388804 "Crow JF. Hardy, Weinberg and language impediments. Genetics. 1999Jul;152(3):821-5. PubMed PMID: 10388804; PubMed Central PMCID: PMC1460671."
[D'Alessandro 1998]: http://www.ncbi.nlm.nih.gov/pubmed/9872257 "D'Alessandro M, Petraroli R, Ladogana A, Pocchiari M. High incidence ofCreutzfeldt-Jakob disease in rural Calabria, Italy. Lancet. 1998 Dec19-26;352(9145):1989-90. PubMed PMID: 9872257."
[Dossena 2008]: http://www.ncbi.nlm.nih.gov/pubmed/19038218 "Dossena S, Imeri L, Mangieri M, Garofoli A, Ferrari L, Senatore A, Restelli E,Balducci C, Fiordaliso F, Salio M, Bianchi S, Fioriti L, Morbin M, Pincherle A,Marcon G, Villani F, Carli M, Tagliavini F, Forloni G, Chiesa R. Mutant prionprotein expression causes motor and memory deficits and abnormal sleep patternsin a transgenic mouse model. Neuron. 2008 Nov 26;60(4):598-609. doi:10.1016/j.neuron.2008.09.008. PubMed PMID: 19038218."
[Durand 2014a]: http://www.ncbi.nlm.nih.gov/pubmed/24784137 "Durand EY, Eriksson N, McLean CY. Reducing pervasive false-positiveidentical-by-descent segments detected by large-scale pedigree analysis. Mol BiolEvol. 2014 Aug;31(8):2212-22. doi: 10.1093/molbev/msu151. Epub 2014 Apr 30.PubMed PMID: 24784137; PubMed Central PMCID: PMC4104314."
[Farlow 1989]: http://www.ncbi.nlm.nih.gov/pubmed/2812321 "Farlow MR, Yee RD, Dlouhy SR, Conneally PM, Azzarelli B, Ghetti B.Gerstmann-Sträussler-Scheinker disease. I. Extending the clinical spectrum.Neurology. 1989 Nov;39(11):1446-52. PubMed PMID: 2812321."
[Finckh 2000]: http://www.ncbi.nlm.nih.gov/pubmed/10631141 "Finckh U, Müller-Thomsen T, Mann U, Eggers C, Marksteiner J, Meins W, Binetti G, Alberici A, Hock C, Nitsch RM, Gal A. High prevalence of pathogenic mutations in patients with early-onset dementia detected by sequence analyses of fourdifferent genes. Am J Hum Genet. 2000 Jan;66(1):110-7. PubMed PMID: 10631141;PubMed Central PMCID: PMC1288316."
[Finckh 2000]: http://www.ncbi.nlm.nih.gov/pubmed/10631141 "Finckh U, Müller-Thomsen T, Mann U, Eggers C, Marksteiner J, Meins W, Binetti G, Alberici A, Hock C, Nitsch RM, Gal A. High prevalence of pathogenic mutations in patients with early-onset dementia detected by sequence analyses of fourdifferent genes. Am J Hum Genet. 2000 Jan;66(1):110-7. PubMed PMID: 10631141;PubMed Central PMCID: PMC1288316."
[Fischer 1996]: http://www.ncbi.nlm.nih.gov/pubmed/8635458 "Fischer M, Rülicke T, Raeber A, Sailer A, Moser M, Oesch B, Brandner S, AguzziA, Weissmann C. Prion protein (PrP) with amino-proximal deletions restoringsusceptibility of PrP knockout mice to scrapie. EMBO J. 1996 Mar15;15(6):1255-64. PubMed PMID: 8635458; PubMed Central PMCID: PMC450028."
[Flannick 2013]: http://www.ncbi.nlm.nih.gov/pubmed/24097065 "Flannick J, Beer NL, Bick AG, Agarwala V, Molnes J, Gupta N, Burtt NP, Florez JC, Meigs JB, Taylor H, Lyssenko V, Irgens H, Fox E, Burslem F, Johansson S,Brosnan MJ, Trimmer JK, Newton-Cheh C, Tuomi T, Molven A, Wilson JG, O'DonnellCJ, Kathiresan S, Hirschhorn JN, Njølstad PR, Rolph T, Seidman JG, Gabriel S, CoxDR, Seidman CE, Groop L, Altshuler D. Assessing the phenotypic effects in thegeneral population of rare variants in genes for a dominant Mendelian form ofdiabetes. Nat Genet. 2013 Nov;45(11):1380-5. doi: 10.1038/ng.2794. Epub 2013 Oct 6. PubMed PMID: 24097065; PubMed Central PMCID: PMC4051627."
[Goldfarb 1990]: http://www.ncbi.nlm.nih.gov/pubmed/1975415 "Goldfarb LG, Korczyn AD, Brown P, Chapman J, Gajdusek DC. Mutation in codon200 of scrapie amyloid precursor gene linked to Creutzfeldt-Jakob disease inSephardic Jews of Libyan and non-Libyan origin. Lancet. 1990 Sep8;336(8715):637-8. PubMed PMID: 1975415."
[Goldfarb 1991a]: http://www.ncbi.nlm.nih.gov/pubmed/1671440 "Goldfarb LG, Haltia M, Brown P, Nieto A, Kovanen J, McCombie WR, Trapp S,Gajdusek DC. New mutation in scrapie amyloid precursor gene (at codon 178) inFinnish Creutzfeldt-Jakob kindred. Lancet. 1991 Feb 16;337(8738):425. PubMedPMID: 1671440."
[Goldfarb 1991b]: http://www.ncbi.nlm.nih.gov/pubmed/1683708 "Goldfarb LG, Brown P, McCombie WR, Goldgaber D, Swergold GD, Wills PR,Cervenakova L, Baron H, Gibbs CJ Jr, Gajdusek DC. Transmissible familial Creutzfeldt-Jakob disease associated with five, seven, and eight extraoctapeptide coding repeats in the PRNP gene. Proc Natl Acad Sci U S A. 1991 Dec1;88(23):10926-30. PubMed PMID: 1683708; PubMed Central PMCID: PMC53045."
[Goldfarb 1992]: http://www.ncbi.nlm.nih.gov/pubmed/1439789 "Goldfarb LG, Petersen RB, Tabaton M, Brown P, LeBlanc AC, Montagna P, CortelliP, Julien J, Vital C, Pendelbury WW, et al. Fatal familial insomnia and familial Creutzfeldt-Jakob disease: disease phenotype determined by a DNA polymorphism.Science. 1992 Oct 30;258(5083):806-8. PubMed PMID: 1439789."
[Goldgaber 1989]: http://www.ncbi.nlm.nih.gov/pubmed/2572450 "Goldgaber D, Goldfarb LG, Brown P, Asher DM, Brown WT, Lin S, Teener JW,Feinstone SM, Rubenstein R, Kascsak RJ, et al. Mutations in familialCreutzfeldt-Jakob disease and Gerstmann-Sträussler-Scheinker's syndrome. ExpNeurol. 1989 Nov;106(2):204-6. PubMed PMID: 2572450."
[Goldwurm 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17215492 "Goldwurm S, Zini M, Mariani L, Tesei S, Miceli R, Sironi F, Clementi M,Bonifati V, Pezzoli G. Evaluation of LRRK2 G2019S penetrance: relevance forgenetic counseling in Parkinson disease. Neurology. 2007 Apr 3;68(14):1141-3.Epub 2007 Jan 10. PubMed PMID: 17215492."
[Grasbon-Frodl 2004]: http://www.ncbi.nlm.nih.gov/pubmed/15558291 "Grasbon-Frodl E, Lorenz H, Mann U, Nitsch RM, Windl O, Kretzschmar HA. Loss ofglycosylation associated with the T183A mutation in human prion disease. ActaNeuropathol. 2004 Dec;108(6):476-84. Epub 2004 Oct 26. PubMed PMID: 15558291."
[Heinemann 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18443555 "Heinemann U, Krasnianski A, Meissner B, Grasbon-Frodl EM, Kretzschmar HA, ZerrI. Novel PRNP mutation in a patient with a slow progressive dementia syndrome.Med Sci Monit. 2008 May;14(5):CS41-43. PubMed PMID: 18443555."
[Hill 2006]: http://www.ncbi.nlm.nih.gov/pubmed/16415305 "Hill AF, Joiner S, Beck JA, Campbell TA, Dickinson A, Poulter M, Wadsworth JD,Collinge J. Distinct glycoform ratios of protease resistant prion proteinassociated with PRNP point mutations. Brain. 2006 Mar;129(Pt 3):676-85. Epub 2006Jan 16. PubMed PMID: 16415305."
[Hilton 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19187063 "Hilton DA, Head MW, Singh VK, Bishop M, Ironside JW. Familial prion diseasewith a novel serine to isoleucine mutation at codon 132 of prion protein gene(PRNP). Neuropathol Appl Neurobiol. 2009 Feb;35(1):111-5. doi:10.1111/j.1365-2990.2008.00964.x. PubMed PMID: 19187063."
[Hinnell 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21282596 "Hinnell C, Coulthart MB, Jansen GH, Cashman NR, Lauzon J, Clark A, Costello F,White C, Midha R, Wiebe S, Furtado S. Gerstmann-Straussler-Scheinker disease due to a novel prion protein gene mutation. Neurology. 2011 Feb 1;76(5):485-7. doi:10.1212/WNL.0b013e31820a0ab2. PubMed PMID: 21282596."
[Hitoshi 1993]: http://www.ncbi.nlm.nih.gov/pubmed/8138811 "Hitoshi S, Nagura H, Yamanouchi H, Kitamoto T. Double mutations at codon 180and codon 232 of the PRNP gene in an apparently sporadic case ofCreutzfeldt-Jakob disease. J Neurol Sci. 1993 Dec 15;120(2):208-12. PubMed PMID: 8138811."
[Hofman 2015]: http://www.ncbi.nlm.nih.gov/pubmed/26386597 "Hofman A, Brusselle GG, Darwish Murad S, van Duijn CM, Franco OH, Goedegebure A, Ikram MA, Klaver CC, Nijsten TE, Peeters RP, Stricker BH, Tiemeier HW,Uitterlinden AG, Vernooij MW. The Rotterdam Study: 2016 objectives and designupdate. Eur J Epidemiol. 2015 Aug;30(8):661-708. doi: 10.1007/s10654-015-0082-x. Epub 2015 Sep 19. PubMed PMID: 26386597; PubMed Central PMCID: PMC4579264."
[Holman 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20049325 "Holman RC, Belay ED, Christensen KY, Maddox RA, Minino AM, Folkema AM,Haberling DL, Hammett TA, Kochanek KD, Sejvar JJ, Schonberger LB. Human priondiseases in the United States. PLoS One. 2010 Jan 1;5(1):e8521. doi:10.1371/journal.pone.0008521. PubMed PMID: 20049325; PubMed Central PMCID:PMC2797136."
[Hoque 1996]: http://www.ncbi.nlm.nih.gov/pubmed/8922054 "Hoque MZ, Kitamoto T, Furukawa H, Muramoto T, Tateishi J. Mutation in theprion protein gene at codon 232 in Japanese patients with Creutzfeldt-Jakobdisease: a clinicopathological, immunohistochemical and transmission study. Acta Neuropathol. 1996 Nov;92(5):441-6. PubMed PMID: 8922054."
[Hsiao 1989]: http://www.ncbi.nlm.nih.gov/pubmed/2564168 "Hsiao K, Baker HF, Crow TJ, Poulter M, Owen F, Terwilliger JD, Westaway D, OttJ, Prusiner SB. Linkage of a prion protein missense variant toGerstmann-Sträussler syndrome. Nature. 1989 Mar 23;338(6213):342-5. PubMed PMID: 2564168."
[Hsiao 1990]: http://www.ncbi.nlm.nih.gov/pubmed/1980379 "Hsiao KK, Scott M, Foster D, Groth DF, DeArmond SJ, Prusiner SB. Spontaneousneurodegeneration in transgenic mice with mutant prion protein. Science. 1990 Dec14;250(4987):1587-90. PubMed PMID: 1980379."
[Hsiao 1991a]: http://www.ncbi.nlm.nih.gov/pubmed/1674116 "Hsiao KK, Cass C, Schellenberg GD, Bird T, Devine-Gage E, Wisniewski H,Prusiner SB. A prion protein variant in a family with the telencephalic form ofGerstmann-Sträussler-Scheinker syndrome. Neurology. 1991 May;41(5):681-4. PubMed PMID: 1674116."
[Hsiao 1991b]: http://www.ncbi.nlm.nih.gov/pubmed/2008182 "Hsiao K, Meiner Z, Kahana E, Cass C, Kahana I, Avrahami D, Scarlato G,Abramsky O, Prusiner SB, Gabizon R. Mutation of the prion protein in Libyan Jews with Creutzfeldt-Jakob disease. N Engl J Med. 1991 Apr 18;324(16):1091-7. PubMed PMID: 2008182."
[Hsiao 1992]: http://www.ncbi.nlm.nih.gov/pubmed/1363810 "Hsiao K, Dlouhy SR, Farlow MR, Cass C, Da Costa M, Conneally PM, Hodes ME,Ghetti B, Prusiner SB. Mutant prion proteins in Gerstmann-Sträussler-Scheinkerdisease with neurofibrillary tangles. Nat Genet. 1992 Apr;1(1):68-71. PubMedPMID: 1363810."
[Jackson 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19709627 "Jackson WS, Borkowski AW, Faas H, Steele AD, King OD, Watson N, Jasanoff A,Lindquist S. Spontaneous generation of prion infectivity in fatal familialinsomnia knockin mice. Neuron. 2009 Aug 27;63(4):438-50. doi:10.1016/j.neuron.2009.07.026. PubMed PMID: 19709627; PubMed Central PMCID:PMC2775465."
[Jackson 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23959875 "Jackson WS, Borkowski AW, Watson NE, King OD, Faas H, Jasanoff A, Lindquist S.Profoundly different prion diseases in knock-in mice carrying single PrP codonsubstitutions associated with human diseases. Proc Natl Acad Sci U S A. 2013 Sep 3;110(36):14759-64. doi: 10.1073/pnas.1312006110. Epub 2013 Aug 19. PubMed PMID: 23959875; PubMed Central PMCID: PMC3767526."
[Jansen 1998]: http://www.ncbi.nlm.nih.gov/pubmed/22558438 "Jansen C, Parchi P, Capellari S, Ibrahim-Verbaas CA, Schuur M, Strammiello R, Corrado P, Bishop MT, van Gool WA, Verbeek MM, Baas F, van Saane W, Spliet WG,Jansen GH, van Duijn CM, Rozemuller AJ. Human prion diseases in the Netherlands(1998-2009): clinical, genetic and molecular aspects. PLoS One. 2012;7(4):e36333.doi: 10.1371/journal.pone.0036333. Epub 2012 Apr 30. PubMed PMID: 22558438;PubMed Central PMCID: PMC3340342."
[Jansen 2010]: http://www.ncbi.nlm.nih.gov/pubmed/19911184 "Jansen C, Parchi P, Capellari S, Vermeij AJ, Corrado P, Baas F, Strammiello R,van Gool WA, van Swieten JC, Rozemuller AJ. Prion protein amyloidosis withdivergent phenotype associated with two novel nonsense mutations in PRNP. ActaNeuropathol. 2010 Feb;119(2):189-97. doi: 10.1007/s00401-009-0609-x. Epub 2009Nov 13. PubMed PMID: 19911184; PubMed Central PMCID: PMC2808512."
[Jansen 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22558438 "Jansen C, Parchi P, Capellari S, Ibrahim-Verbaas CA, Schuur M, Strammiello R,  Corrado P, Bishop MT, van Gool WA, Verbeek MM, Baas F, van Saane W, Spliet WG, Jansen GH, van Duijn CM, Rozemuller AJ. Human prion diseases in the Netherlands (1998-2009): clinical, genetic and molecular aspects. PLoS One. 2012;7(4):e36333. doi: 10.1371/journal.pone.0036333. Epub 2012 Apr 30. PubMed PMID: 22558438; PubMed Central PMCID: PMC3340342."
[Jayadev 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21416485 "Jayadev S, Nochlin D, Poorkaj P, Steinbart EJ, Mastrianni JA, Montine TJ,Ghetti B, Schellenberg GD, Bird TD, Leverenz JB. Familial prion disease withAlzheimer disease-like tau pathology and clinical phenotype. Ann Neurol. 2011Apr;69(4):712-20. doi: 10.1002/ana.22264. Epub 2011 Mar 17. PubMed PMID:21416485; PubMed Central PMCID: PMC3114566."
[Jeong 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20497338 "Jeong BH, Jeon YC, Lee YJ, Cho HJ, Park SJ, Chung DI, Kim J, Kim SH, Kim HT,Choi EK, Choi KC, Carp RI, Kim YS. Creutzfeldt-Jakob disease with the V203Imutation and M129V polymorphism of the prion protein gene (PRNP) and a 17 kDaprion protein fragment. Neuropathol Appl Neurobiol. 2010 Oct;36(6):558-63. doi:10.1111/j.1365-2990.2010.01094.x. PubMed PMID: 20497338."
[Jones 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24814844 "Jones M, Odunsi S, du Plessis D, Vincent A, Bishop M, Head MW, Ironside JW,Gow D. Gerstmann-Straüssler-Scheinker disease: novel PRNP mutation andVGKC-complex antibodies. Neurology. 2014 Jun 10;82(23):2107-11. doi:10.1212/WNL.0000000000000500. Epub 2014 May 9. PubMed PMID: 24814844; PubMedCentral PMCID: PMC4118501."
[Karapetyan & Sferrazza 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23576755 "Karapetyan YE, Sferrazza GF, Zhou M, Ottenberg G, Spicer T, Chase P, FallahiM, Hodder P, Weissmann C, Lasmézas CI. Unique drug screening approach for priondiseases identifies tacrolimus and astemizole as antiprion agents. Proc Natl AcadSci U S A. 2013 Apr 23;110(17):7044-9. doi: 10.1073/pnas.1303510110. Epub 2013Apr 1. PubMed PMID: 23576755; PubMed Central PMCID: PMC3637718."
[Kaski 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21616973 "Kaski DN, Pennington C, Beck J, Poulter M, Uphill J, Bishop MT, Linehan JM,O'Malley C, Wadsworth JD, Joiner S, Knight RS, Ironside JW, Brandner S, Collinge J, Mead S. Inherited prion disease with 4-octapeptide repeat insertion: diseaserequires the interaction of multiple genetic risk factors. Brain. 2011 Jun;134(Pt6):1829-38. doi: 10.1093/brain/awr079. PubMed PMID: 21616973."
[Kim 2013]: http://www.ncbi.nlm.nih.gov/pubmed/24330864 "Kim MO, Cali I, Oehler A, Fong JC, Wong K, See T, Katz JS, Gambetti P,Bettcher BM, Dearmond SJ, Geschwind MD. Genetic CJD with a novel E200G mutationin the prion protein gene and comparison with E200K mutation cases. ActaNeuropathol Commun. 2013 Dec 12;1:80. doi: 10.1186/2051-5960-1-80. PubMed PMID:24330864; PubMed Central PMCID: PMC3880091."
[Kirov 2014]: http://www.ncbi.nlm.nih.gov/pubmed/23992924 "Kirov G, Rees E, Walters JT, Escott-Price V, Georgieva L, Richards AL,Chambert KD, Davies G, Legge SE, Moran JL, McCarroll SA, O'Donovan MC, Owen MJ.The penetrance of copy number variations for schizophrenia and developmentaldelay. Biol Psychiatry. 2014 Mar 1;75(5):378-85. doi:10.1016/j.biopsych.2013.07.022. Epub 2013 Aug 28. PubMed PMID: 23992924; PubMedCentral PMCID: PMC4229045."
[Kitamoto 1993]: http://www.ncbi.nlm.nih.gov/pubmed/8097911 "Kitamoto T, Iizuka R, Tateishi J. An amber mutation of prion protein inGerstmann-Sträussler syndrome with mutant PrP plaques. Biochem Biophys ResCommun. 1993 Apr 30;192(2):525-31. PubMed PMID: 8097911."
[Klug 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23965290 "Klug GM, Wand H, Simpson M, Boyd A, Law M, Masters CL, Matěj R, Howley R,Farrell M, Breithaupt M, Zerr I, van Duijn C, Ibrahim-Verbaas C, Mackenzie J,Will RG, Brandel JP, Alperovitch A, Budka H, Kovacs GG, Jansen GH, Coulthard M,Collins SJ. Intensity of human prion disease surveillance predicts observeddisease incidence. J Neurol Neurosurg Psychiatry. 2013 Dec;84(12):1372-7. doi:10.1136/jnnp-2012-304820. Epub 2013 Aug 21. PubMed PMID: 23965290."
[Komatsu 2014]: http://www.ncbi.nlm.nih.gov/pubmed/25495585 "Komatsu J, Sakai K, Hamaguchi T, Sugiyama Y, Iwasa K, Yamada M.Creutzfeldt-Jakob disease associated with a V203I homozygous mutation in theprion protein gene. Prion. 2014;8(5):336-8. doi: 10.4161/19336896.2014.971569.PubMed PMID: 25495585; PubMed Central PMCID: PMC4601383."
[Kotta 2006]: http://www.ncbi.nlm.nih.gov/pubmed/17129366 "Kotta K, Paspaltsis I, Bostantjopoulou S, Latsoudis H, Plaitakis A, Kazis D,Collinge J, Sklaviadis T. Novel mutation of the PRNP gene of a clinical CJD case.BMC Infect Dis. 2006 Nov 27;6:169. PubMed PMID: 17129366; PubMed Central PMCID:PMC1693557."
[Kovacs 2005]: http://www.ncbi.nlm.nih.gov/pubmed/16187142 "Kovács GG, Puopolo M, Ladogana A, Pocchiari M, Budka H, van Duijn C, CollinsSJ, Boyd A, Giulivi A, Coulthart M, Delasnerie-Laupretre N, Brandel JP, Zerr I,Kretzschmar HA, de Pedro-Cuesta J, Calero-Lara M, Glatzel M, Aguzzi A, Bishop M, Knight R, Belay G, Will R, Mitrova E; EUROCJD. Genetic prion disease: the EUROCJDexperience. Hum Genet. 2005 Nov;118(2):166-74. Epub 2005 Nov 15. PubMed PMID:16187142."
[Krasemann 1995]: http://www.ncbi.nlm.nih.gov/pubmed/8750875 "Krasemann S, Zerr I, Weber T, Poser S, Kretzschmar H, Hunsmann G, Bodemer W.Prion disease associated with a novel nine octapeptide repeat insertion in thePRNP gene. Brain Res Mol Brain Res. 1995 Dec 1;34(1):173-6. PubMed PMID: 8750875."
[Krebs 2005]: http://www.ncbi.nlm.nih.gov/pubmed/15776279 "Krebs B, Lederer RM, Windl O, Grasbon-Frodl EM, Zerr I, Kretzschmar HA.Creutzfeldt-Jakob disease associated with an R148H mutation of the prion protein gene. Neurogenetics. 2005 May;6(2):97-100. Epub 2005 Mar 18. PubMed PMID:15776279."
[Kumar 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21911696 "Kumar N, Boeve BF, Boot BP, Orr CF, Duffy J, Woodruff BK, Nair AK, Ellison J, Kuntz K, Kantarci K, Jack CR Jr, Westmoreland BF, Fields JA, Baker M, Rademakers R, Parisi JE, Dickson DW. Clinical characterization of a kindred with a novel12-octapeptide repeat insertion in the prion protein gene. Arch Neurol. 2011Sep;68(9):1165-70. doi: 10.1001/archneurol.2011.187. PubMed PMID: 21911696;PubMed Central PMCID: PMC3326586."
[Ladogana 2005]: http://www.ncbi.nlm.nih.gov/pubmed/15883322 "Ladogana A, Puopolo M, Poleggi A, Almonti S, Mellina V, Equestre M, Pocchiari M. High incidence of genetic human transmissible spongiform encephalopathies inItaly. Neurology. 2005 May 10;64(9):1592-7. PubMed PMID: 15883322."
[Landrum 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24234437 "Landrum MJ, Lee JM, Riley GR, Jang W, Rubinstein WS, Church DM, Maglott DR.ClinVar: public archive of relationships among sequence variation and humanphenotype. Nucleic Acids Res. 2014 Jan;42(Database issue):D980-5. doi:10.1093/nar/gkt1113. Epub 2013 Nov 14. PubMed PMID: 24234437; PubMed CentralPMCID: PMC3965032."
[Laplanche 1995]: http://www.ncbi.nlm.nih.gov/pubmed/7655470 "Laplanche JL, Delasnerie-Lauprêtre N, Brandel JP, Dussaucy M, Chatelain J,Launay JM. Two novel insertions in the prion protein gene in patients withlate-onset dementia. Hum Mol Genet. 1995 Jun;4(6):1109-11. PubMed PMID: 7655470."
[Laplanche 1999]: http://www.ncbi.nlm.nih.gov/pubmed/10581230 "Laplanche JL, Hachimi KH, Durieux I, Thuillet P, Defebvre L,Delasnerie-Lauprêtre N, Peoc'h K, Foncin JF, Destée A. Prominent psychiatricfeatures and early onset in an inherited prion disease with a new insertionalmutation in the prion protein gene. Brain. 1999 Dec;122 ( Pt 12):2375-86. PubMed PMID: 10581230."
[Lee 1999]: http://www.ncbi.nlm.nih.gov/pubmed/10090891 "Lee HS, Sambuughin N, Cervenakova L, Chapman J, Pocchiari M, Litvak S, Qi HY, Budka H, del Ser T, Furukawa H, Brown P, Gajdusek DC, Long JC, Korczyn AD,Goldfarb LG. Ancestral origins and worldwide distribution of the PRNP 200Kmutation causing familial Creutzfeldt-Jakob disease. Am J Hum Genet. 1999Apr;64(4):1063-70. PubMed PMID: 10090891; PubMed Central PMCID: PMC1377830."
[Lewis 2003]: http://www.ncbi.nlm.nih.gov/pubmed/12771252 "Lewis V, Collins S, Hill AF, Boyd A, McLean CA, Smith M, Masters CL. Novelprion protein insert mutation associated with prolonged neurodegenerativeillness. Neurology. 2003 May 27;60(10):1620-4. PubMed PMID: 12771252."
[Liu 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20028338 "Liu Z, Jia L, Piao Y, Lu D, Wang F, Lv H, Lu Y, Jia J. Creutzfeldt-Jakobdisease with PRNP G114V mutation in a Chinese family. Acta Neurol Scand. 2010Jun;121(6):377-83. doi: 10.1111/j.1600-0404.2009.01236.x. Epub 2009 Dec 17.PubMed PMID: 20028338."
[MacArthur 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24759409 "MacArthur DG, Manolio TA, Dimmock DP, Rehm HL, Shendure J, Abecasis GR, Adams DR, Altman RB, Antonarakis SE, Ashley EA, Barrett JC, Biesecker LG, Conrad DF,Cooper GM, Cox NJ, Daly MJ, Gerstein MB, Goldstein DB, Hirschhorn JN, Leal SM,Pennacchio LA, Stamatoyannopoulos JA, Sunyaev SR, Valle D, Voight BF, Winckler W,Gunter C. Guidelines for investigating causality of sequence variants in humandisease. Nature. 2014 Apr 24;508(7497):469-76. doi: 10.1038/nature13127. PubMedPMID: 24759409; PubMed Central PMCID: PMC4180223."
[Mallucci 2003]: http://www.ncbi.nlm.nih.gov/pubmed/14593181 "Mallucci G, Dickinson A, Linehan J, Klöhn PC, Brandner S, Collinge J.Depleting neuronal PrP in prion infection prevents disease and reversesspongiosis. Science. 2003 Oct 31;302(5646):871-4. PubMed PMID: 14593181."
[Masters 1979]: http://www.ncbi.nlm.nih.gov/pubmed/371520 "Masters CL, Harris JO, Gajdusek DC, Gibbs CJ Jr, Bernoulli C, Asher DM.Creutzfeldt-Jakob disease: patterns of worldwide occurrence and the significance of familial and sporadic clustering. Ann Neurol. 1979 Feb;5(2):177-88. PubMedPMID: 371520."
[Mastrianni 1995]: http://www.ncbi.nlm.nih.gov/pubmed/7501157 "Mastrianni JA, Curtis MT, Oberholtzer JC, Da Costa MM, DeArmond S, PrusinerSB, Garbern JY. Prion disease (PrP-A117V) presenting with ataxia instead ofdementia. Neurology. 1995 Nov;45(11):2042-50. PubMed PMID: 7501157."
[Mastrianni 1996]: http://www.ncbi.nlm.nih.gov/pubmed/8909447 "Mastrianni JA, Iannicola C, Myers RM, DeArmond S, Prusiner SB. Mutation of theprion protein gene at codon 208 in familial Creutzfeldt-Jakob disease. Neurology.1996 Nov;47(5):1305-12. PubMed PMID: 8909447."
[Mastrianni 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20216075 "Mastrianni JA. The genetics of prion diseases. Genet Med. 2010Apr;12(4):187-95. doi: 10.1097/GIM.0b013e3181cd7374. Review. PubMed PMID:20216075."
[Matej 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22488860 "Matěj R, Kovacs GG, Johanidesová S, Keller J, Matějčková M, Nováková J, Sigut V, Keller O, Rusina R. Genetic Creutzfeldt-Jakob disease with R208H mutationpresenting as progressive supranuclear palsy. Mov Disord. 2012 Apr;27(4):476-9.doi: 10.1002/mds.24002. PubMed PMID: 22488860."
[Matsuzono 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23577609 "Matsuzono K, Ikeda Y, Liu W, Kurata T, Deguchi S, Deguchi K, Abe K. A novelfamilial prion disease causing pan-autonomic-sensory neuropathy and cognitiveimpairment. Eur J Neurol. 2013 May;20(5):e67-9. doi: 10.1111/ene.12089. PubMedPMID: 23577609."
[Mead & Reilly 2015]: http://www.ncbi.nlm.nih.gov/pubmed/25623792 "Mead S, Reilly MM. A new prion disease: relationship with central andperipheral amyloidoses. Nat Rev Neurol. 2015 Feb;11(2):90-7. doi:10.1038/nrneurol.2014.263. Epub 2015 Jan 27. Review. PubMed PMID: 25623792."
[Mead 2006a]: http://www.ncbi.nlm.nih.gov/pubmed/16391566 "Mead S. Prion disease genetics. Eur J Hum Genet. 2006 Mar;14(3):273-81.Review. PubMed PMID: 16391566."
[Mead 2006b]: http://www.ncbi.nlm.nih.gov/pubmed/16923955 "Mead S, Poulter M, Beck J, Webb TE, Campbell TA, Linehan JM, Desbruslais M,Joiner S, Wadsworth JD, King A, Lantos P, Collinge J. Inherited prion diseasewith six octapeptide repeat insertional mutation--molecular analysis ofphenotypic heterogeneity. Brain. 2006 Sep;129(Pt 9):2297-317. PubMed PMID:16923955."
[Mead 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22210626 "Mead S, Uphill J, Beck J, Poulter M, Campbell T, Lowe J, Adamson G, Hummerich H, Klopp N, Rückert IM, Wichmann HE, Azazi D, Plagnol V, Pako WH, Whitfield J,Alpers MP, Whittaker J, Balding DJ, Zerr I, Kretzschmar H, Collinge J.Genome-wide association study in multiple human prion diseases suggests geneticrisk factors additional to PRNP. Hum Mol Genet. 2012 Apr 15;21(8):1897-906. doi: 10.1093/hmg/ddr607. Epub 2011 Dec 30. PubMed PMID: 22210626; PubMed CentralPMCID: PMC3313791."
[Mead 2013]: http://www.ncbi.nlm.nih.gov/pubmed/24224623 "Mead S, Gandhi S, Beck J, Caine D, Gajulapalli D, Carswell C, Hyare H, Joiner S, Ayling H, Lashley T, Linehan JM, Al-Doujaily H, Sharps B, Revesz T, SandbergMK, Reilly MM, Koltzenburg M, Forbes A, Rudge P, Brandner S, Warren JD, WadsworthJD, Wood NW, Holton JL, Collinge J. A novel prion disease associated withdiarrhea and autonomic neuropathy. N Engl J Med. 2013 Nov 14;369(20):1904-14.doi: 10.1056/NEJMoa1214747. Erratum in: N Engl J Med. 2014 Jan 9;370(2):188.Gallujipali, Dillip [corrected to Gajulapalli, Dilip]. PubMed PMID: 24224623;PubMed Central PMCID: PMC3863770."
[Medori 1992]: http://www.ncbi.nlm.nih.gov/pubmed/1346338 "Medori R, Tritschler HJ, LeBlanc A, Villare F, Manetto V, Chen HY, Xue R, LealS, Montagna P, Cortelli P, et al. Fatal familial insomnia, a prion disease with amutation at codon 178 of the prion protein gene. N Engl J Med. 1992 Feb13;326(7):444-9. PubMed PMID: 1346338."
[Medori 1992]: http://www.ncbi.nlm.nih.gov/pubmed/1347910 "Medori R, Montagna P, Tritschler HJ, LeBlanc A, Cortelli P, Tinuper P,Lugaresi E, Gambetti P. Fatal familial insomnia: a second kindred with mutationof prion protein gene at codon 178. Neurology. 1992 Mar;42(3 Pt 1):669-70. PubMedPMID: 1347910."
[Minikel 2014]: http://www.ncbi.nlm.nih.gov/pubmed/25279981 "Minikel EV, Zerr I, Collins SJ, Ponto C, Boyd A, Klug G, Karch A, Kenny J,Collinge J, Takada LT, Forner S, Fong JC, Mead S, Geschwind MD. Ascertainmentbias causes false signal of anticipation in genetic prion disease. Am J HumGenet. 2014 Oct 2;95(4):371-82. doi: 10.1016/j.ajhg.2014.09.003. PubMed PMID:25279981; PubMed Central PMCID: PMC4185115."
[Mitrova & Belay 2002]: http://www.ncbi.nlm.nih.gov/pubmed/12197632 "Mitrová E, Belay G. Creutzfeldt-Jakob disease with E200K mutation in Slovakia:characterization and development. Acta Virol. 2002;46(1):31-9. PubMed PMID:12197632."
[Moore 2001]: http://www.ncbi.nlm.nih.gov/pubmed/11593450 "Moore RC, Xiang F, Monaghan J, Han D, Zhang Z, Edström L, Anvret M, PrusinerSB. Huntington disease phenocopy is a familial prion disease. Am J Hum Genet.2001 Dec;69(6):1385-8. Epub 2001 Oct 9. PubMed PMID: 11593450; PubMed CentralPMCID: PMC1235549."
[Mouillet-Richard 1999]: http://www.ncbi.nlm.nih.gov/pubmed/10526198 "Mouillet-Richard S, Teil C, Lenne M, Hugon S, Taleb O, Laplanche JL. Mutation at codon 210 (V210I) of the prion protein gene in a North African patient withCreutzfeldt-Jakob disease. J Neurol Sci. 1999 Oct 15;168(2):141-4. PubMed PMID:10526198."
[Munoz-Nieto 2013]: http://www.ncbi.nlm.nih.gov/pubmed/22763467 "Muñoz-Nieto M, Ramonet N, López-Gastón JI, Cuadrado-Corrales N, Calero O,Díaz-Hurtado M, Ipiens JR, Ramón y Cajal S, de Pedro-Cuesta J, Calero M. A novel mutation I215V in the PRNP gene associated with Creutzfeldt-Jakob and Alzheimer'sdiseases in three patients with divergent clinical phenotypes. J Neurol. 2013Jan;260(1):77-84. doi: 10.1007/s00415-012-6588-1. Epub 2012 Jul 5. PubMed PMID:22763467."
[Nazor Friberg 2012]: http://www.ncbi.nlm.nih.gov/pubmed/23344724 "Nazor Friberg K, Hung G, Wancewicz E, Giles K, Black C, Freier S, Bennett F,Dearmond SJ, Freyman Y, Lessard P, Ghaemmaghami S, Prusiner SB. IntracerebralInfusion of Antisense Oligonucleotides Into Prion-infected Mice. Mol Ther NucleicAcids. 2012 Feb 7;1:e9. doi: 10.1038/mtna.2011.6. PubMed PMID: 23344724; PubMedCentral PMCID: PMC3381600."
[Nazor Friberg 2012]: http://www.ncbi.nlm.nih.gov/pubmed/23344724 "Nazor Friberg K, Hung G, Wancewicz E, Giles K, Black C, Freier S, Bennett F,Dearmond SJ, Freyman Y, Lessard P, Ghaemmaghami S, Prusiner SB. IntracerebralInfusion of Antisense Oligonucleotides Into Prion-infected Mice. Mol Ther NucleicAcids. 2012 Feb 7;1:e9. doi: 10.1038/mtna.2011.6. PubMed PMID: 23344724; PubMedCentral PMCID: PMC3381600."
[Nishida 2004]: http://www.ncbi.nlm.nih.gov/pubmed/15557533 "Nishida Y, Sodeyama N, Toru Y, Toru S, Kitamoto T, Mizusawa H.Creutzfeldt-Jakob disease with a novel insertion and codon 219 Lys/Lyspolymorphism in PRNP. Neurology. 2004 Nov 23;63(10):1978-9. PubMed PMID:15557533."
[Nitrini 1997]: http://www.ncbi.nlm.nih.gov/pubmed/9266722 "Nitrini R, Rosemberg S, Passos-Bueno MR, da Silva LS, Iughetti P, PapadopoulosM, Carrilho PM, Caramelli P, Albrecht S, Zatz M, LeBlanc A. Familial spongiformencephalopathy associated with a novel prion protein gene mutation. Ann Neurol.1997 Aug;42(2):138-46. PubMed PMID: 9266722."
[Nozaki 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20855418 "Nozaki I, Hamaguchi T, Sanjo N, Noguchi-Shinohara M, Sakai K, Nakamura Y, SatoT, Kitamoto T, Mizusawa H, Moriwaka F, Shiga Y, Kuroiwa Y, Nishizawa M, Kuzuhara S, Inuzuka T, Takeda M, Kuroda S, Abe K, Murai H, Murayama S, Tateishi J, Takumi I, Shirabe S, Harada M, Sadakane A, Yamada M. Prospective 10-year surveillance ofhuman prion diseases in Japan. Brain. 2010 Oct;133(10):3043-57. doi:10.1093/brain/awq216. Epub 2010 Sep 20. PubMed PMID: 20855418."
[Owen 1990]: http://www.ncbi.nlm.nih.gov/pubmed/2159587 "Owen F, Poulter M, Shah T, Collinge J, Lofthouse R, Baker H, Ridley R, McVeyJ, Crow TJ. An in-frame insertion in the prion protein gene in familialCreutzfeldt-Jakob disease. Brain Res Mol Brain Res. 1990 Apr;7(3):273-6. PubMedPMID: 2159587."
[Owen 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24713662 "Owen J, Beck J, Campbell T, Adamson G, Gorham M, Thompson A, Smithson S,Rosser E, Rudge P, Collinge J, Mead S. Predictive testing for inherited priondisease: report of 22 years experience. Eur J Hum Genet. 2014 Dec;22(12):1351-6. doi: 10.1038/ejhg.2014.42. Epub 2014 Apr 9. PubMed PMID: 24713662; PubMed CentralPMCID: PMC4091984."
[Panegyres 2001]: http://www.ncbi.nlm.nih.gov/pubmed/11709001 "Panegyres PK, Toufexis K, Kakulas BA, Cernevakova L, Brown P, Ghetti B,Piccardo P, Dlouhy SR. A new PRNP mutation (G131V) associated withGerstmann-Sträussler-Scheinker disease. Arch Neurol. 2001 Nov;58(11):1899-902.PubMed PMID: 11709001."
[Parchi 1999]: http://www.ncbi.nlm.nih.gov/pubmed/10443888 "Parchi P, Giese A, Capellari S, Brown P, Schulz-Schaeffer W, Windl O, Zerr I, Budka H, Kopp N, Piccardo P, Poser S, Rojiani A, Streichemberger N, Julien J,Vital C, Ghetti B, Gambetti P, Kretzschmar H. Classification of sporadicCreutzfeldt-Jakob disease based on molecular and phenotypic analysis of 300subjects. Ann Neurol. 1999 Aug;46(2):224-33. PubMed PMID: 10443888."
[Pastore 2005]: http://www.ncbi.nlm.nih.gov/pubmed/16314483 "Pastore M, Chin SS, Bell KL, Dong Z, Yang Q, Yang L, Yuan J, Chen SG, GambettiP, Zou WQ. Creutzfeldt-Jakob disease (CJD) with a mutation at codon 148 of prion protein gene: relationship with sporadic CJD. Am J Pathol. 2005Dec;167(6):1729-38. PubMed PMID: 16314483; PubMed Central PMCID: PMC1613192."
[Peoc'h 2000]: http://www.ncbi.nlm.nih.gov/pubmed/10790216 "Peoc'h K, Manivet P, Beaudry P, Attane F, Besson G, Hannequin D,Delasnerie-Lauprêtre N, Laplanche JL. Identification of three novel mutations(E196K, V203I, E211Q) in the prion protein gene (PRNP) in inherited priondiseases with Creutzfeldt-Jakob disease phenotype. Hum Mutat. 2000 May;15(5):482.PubMed PMID: 10790216."
[Peoc'h 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22965875 "Peoc'h K, Levavasseur E, Delmont E, De Simone A, Laffont-Proust I, Privat N,Chebaro Y, Chapuis C, Bedoucha P, Brandel JP, Laquerriere A, Kemeny JL, Hauw JJ, Borg M, Rezaei H, Derreumaux P, Laplanche JL, Haïk S. Substitutions at residue211 in the prion protein drive a switch between CJD and GSS syndrome, a newmechanism governing inherited neurodegenerative disorders. Hum Mol Genet. 2012Dec 15;21(26):5417-28. doi: 10.1093/hmg/dds377. Epub 2012 Sep 10. Erratum in: HumMol Genet. 2014 Apr 15;23(8):2234. PubMed PMID: 22965875."
[Piccardo 1998]: http://www.ncbi.nlm.nih.gov/pubmed/9786248 "Piccardo P, Dlouhy SR, Lievens PM, Young K, Bird TD, Nochlin D, Dickson DW,Vinters HV, Zimmerman TR, Mackenzie IR, Kish SJ, Ang LC, De Carli C, Pocchiari M,Brown P, Gibbs CJ Jr, Gajdusek DC, Bugiani O, Ironside J, Tagliavini F, Ghetti B.Phenotypic variability of Gerstmann-Sträussler-Scheinker disease is associatedwith prion protein heterogeneity. J Neuropathol Exp Neurol. 1998Oct;57(10):979-88. PubMed PMID: 9786248."
[Pietrini 2003]: http://www.ncbi.nlm.nih.gov/pubmed/14610142 "Pietrini V, Puoti G, Limido L, Rossi G, Di Fede G, Giaccone G, Mangieri M,Tedeschi F, Bondavalli A, Mancia D, Bugiani O, Tagliavini F. Creutzfeldt-Jakobdisease with a novel extra-repeat insertional mutation in the PRNP gene.Neurology. 2003 Nov 11;61(9):1288-91. PubMed PMID: 14610142."
[Pocchiari 1993]: http://www.ncbi.nlm.nih.gov/pubmed/7902693 "Pocchiari M, Salvatore M, Cutruzzolá F, Genuardi M, Allocatelli CT, Masullo C,Macchi G, Alemá G, Galgani S, Xi YG, et al. A new point mutation of the prionprotein gene in Creutzfeldt-Jakob disease. Ann Neurol. 1993 Dec;34(6):802-7.PubMed PMID: 7902693."
[Polymenidou 2011]: http://www.ncbi.nlm.nih.gov/pubmed/20875062 "Polymenidou M, Prokop S, Jung HH, Hewer E, Peretz D, Moos R, Tolnay M, Aguzzi A. Atypical prion protein conformation in familial prion disease with PRNP P105T mutation. Brain Pathol. 2011 Mar;21(2):209-14. doi:10.1111/j.1750-3639.2010.00439.x. Epub 2010 Sep 28. PubMed PMID: 20875062."
[Prusiner 1998]: http://www.ncbi.nlm.nih.gov/pubmed/9811807 "Prusiner SB. Prions. Proc Natl Acad Sci U S A. 1998 Nov 10;95(23):13363-83. Review. PubMed PMID: 9811807; PubMed Central PMCID: PMC33918."
[Pulford 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20559428 "Pulford B, Reim N, Bell A, Veatch J, Forster G, Bender H, Meyerett C, Hafeman S, Michel B, Johnson T, Wyckoff AC, Miele G, Julius C, Kranich J, Schenkel A, DowS, Zabel MD. Liposome-siRNA-peptide complexes cross the blood-brain barrier andsignificantly decrease PrP on neuronal cells and PrP in infected cell cultures.PLoS One. 2010 Jun 14;5(6):e11085. doi: 10.1371/journal.pone.0011085. PubMedPMID: 20559428; PubMed Central PMCID: PMC2885418."
[Pulford 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20559428 "Pulford B, Reim N, Bell A, Veatch J, Forster G, Bender H, Meyerett C, Hafeman S, Michel B, Johnson T, Wyckoff AC, Miele G, Julius C, Kranich J, Schenkel A, DowS, Zabel MD. Liposome-siRNA-peptide complexes cross the blood-brain barrier andsignificantly decrease PrP on neuronal cells and PrP in infected cell cultures.PLoS One. 2010 Jun 14;5(6):e11085. doi: 10.1371/journal.pone.0011085. PubMedPMID: 20559428; PubMed Central PMCID: PMC2885418."
[Puopolo 1993]: http://www.ncbi.nlm.nih.gov/pubmed/12812825 "Puopolo M, Ladogana A, Almonti S, Daude N, Bevivino S, Petraroli R, Poleggi A,Quanguo L, Pocchiari M. Mortality trend from sporadic Creutzfeldt-Jakob disease(CJD) in Italy, 1993-2000. J Clin Epidemiol. 2003 May;56(5):494-9. PubMed PMID:12812825."
[Puopolo 2003]: http://www.ncbi.nlm.nih.gov/pubmed/12812825 "Puopolo M, Ladogana A, Almonti S, Daude N, Bevivino S, Petraroli R, Poleggi A, Quanguo L, Pocchiari M. Mortality trend from sporadic Creutzfeldt-Jakob disease (CJD) in Italy, 1993-2000. J Clin Epidemiol. 2003 May;56(5):494-9. PubMed PMID: 12812825."
[Purcell 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24463508 "Purcell SM, Moran JL, Fromer M, Ruderfer D, Solovieff N, Roussos P,O'Dushlaine C, Chambert K, Bergen SE, Kähler A, Duncan L, Stahl E, Genovese G,Fernández E, Collins MO, Komiyama NH, Choudhary JS, Magnusson PK, Banks E, ShakirK, Garimella K, Fennell T, DePristo M, Grant SG, Haggarty SJ, Gabriel S, ScolnickEM, Lander ES, Hultman CM, Sullivan PF, McCarroll SA, Sklar P. A polygenic burdenof rare disruptive mutations in schizophrenia. Nature. 2014 Feb13;506(7487):185-90. doi: 10.1038/nature12975. Epub 2014 Jan 22. PubMed PMID:24463508; PubMed Central PMCID: PMC4136494."
[Revesz 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19225789 "Revesz T, Holton JL, Lashley T, Plant G, Frangione B, Rostagno A, Ghiso J.Genetics and molecular pathogenesis of sporadic and hereditary cerebral amyloidangiopathies. Acta Neuropathol. 2009 Jul;118(1):115-30. doi:10.1007/s00401-009-0501-8. Epub 2009 Feb 19. Review. Erratum in: ActaNeuropathol. 2009 Aug;118(2):321. PubMed PMID: 19225789; PubMed Central PMCID:PMC2844092."
[Revesz 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19225789 "Revesz T, Holton JL, Lashley T, Plant G, Frangione B, Rostagno A, Ghiso J.Genetics and molecular pathogenesis of sporadic and hereditary cerebral amyloidangiopathies. Acta Neuropathol. 2009 Jul;118(1):115-30. doi:10.1007/s00401-009-0501-8. Epub 2009 Feb 19. Review. Erratum in: ActaNeuropathol. 2009 Aug;118(2):321. PubMed PMID: 19225789; PubMed Central PMCID:PMC2844092."
[Richards 2015]: http://www.ncbi.nlm.nih.gov/pubmed/25741868 "Richards S, Aziz N, Bale S, Bick D, Das S, Gastier-Foster J, Grody WW, HegdeM, Lyon E, Spector E, Voelkerding K, Rehm HL; ACMG Laboratory Quality AssuranceCommittee. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Genet Med. 2015May;17(5):405-24. doi: 10.1038/gim.2015.30. Epub 2015 Mar 5. PubMed PMID:25741868; PubMed Central PMCID: PMC4544753."
[Richt 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17195841 "Richt JA, Kasinathan P, Hamir AN, Castilla J, Sathiyaseelan T, Vargas F,Sathiyaseelan J, Wu H, Matsushita H, Koster J, Kato S, Ishida I, Soto C, Robl JM,Kuroiwa Y. Production of cattle lacking prion protein. Nat Biotechnol. 2007Jan;25(1):132-8. Epub 2006 Dec 31. PubMed PMID: 17195841; PubMed Central PMCID:PMC2813193."
[Ripoll 1993]: http://www.ncbi.nlm.nih.gov/pubmed/8105421 "Ripoll L, Laplanche JL, Salzmann M, Jouvet A, Planques B, Dussaucy M,Chatelain J, Beaudry P, Launay JM. A new point mutation in the prion protein geneat codon 210 in Creutzfeldt-Jakob disease. Neurology. 1993 Oct;43(10):1934-8.PubMed PMID: 8105421."
[Robinson 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21221095 "Robinson JT, Thorvaldsdóttir H, Winckler W, Guttman M, Lander ES, Getz G,Mesirov JP. Integrative genomics viewer. Nat Biotechnol. 2011 Jan;29(1):24-6.doi: 10.1038/nbt.1754. PubMed PMID: 21221095; PubMed Central PMCID: PMC3346182."
[Rodriguez 2005]: http://www.ncbi.nlm.nih.gov/pubmed/15851745 "Rodriguez MM, Peoc'h K, Haïk S, Bouchet C, Vernengo L, Mañana G, Salamano R,Carrasco L, Lenne M, Beaudry P, Launay JM, Laplanche JL. A novel mutation (G114V)in the prion protein gene in a family with inherited prion disease. Neurology.2005 Apr 26;64(8):1455-7. PubMed PMID: 15851745."
[Roeber 2005]: http://www.ncbi.nlm.nih.gov/pubmed/15739100 "Roeber S, Krebs B, Neumann M, Windl O, Zerr I, Grasbon-Frodl EM, KretzschmarHA. Creutzfeldt-Jakob disease in a patient with an R208H mutation of the prionprotein gene (PRNP) and a 17-kDa prion protein fragment. Acta Neuropathol. 2005Apr;109(4):443-8. Epub 2005 Mar 1. PubMed PMID: 15739100."
[Roeber 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18478114 "Roeber S, Grasbon-Frodl EM, Windl O, Krebs B, Xiang W, Vollmert C, Illig T,Schröter A, Arzberger T, Weber P, Zerr I, Kretzschmar HA. Evidence for apathogenic role of different mutations at codon 188 of PRNP. PLoS One. 2008 May14;3(5):e2147. doi: 10.1371/journal.pone.0002147. Erratum in: PLoS ONE.2008;3(5). doi: 10.1371/annotation/3189bbdc-e736-4cb5-b1f2-9a5fb49536c6. PubMedPMID: 18478114; PubMed Central PMCID: PMC2366066."
[Rogaeva 2006]: http://www.ncbi.nlm.nih.gov/pubmed/16831973 "Rogaeva E, Zadikoff C, Ponesse J, Schmitt-Ulms G, Kawarai T, Sato C,Salehi-Rad S, St George-Hyslop P, Lang AE. Childhood onset in familial priondisease with a novel mutation in the PRNP gene. Arch Neurol. 2006Jul;63(7):1016-21. PubMed PMID: 16831973."
[Rowe 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17353478 "Rowe DB, Lewis V, Needham M, Rodriguez M, Boyd A, McLean C, Roberts H, MastersCL, Collins SJ. Novel prion protein gene mutation presenting with subacutePSP-like syndrome. Neurology. 2007 Mar 13;68(11):868-70. PubMed PMID: 17353478."
[Safar 2005]: http://www.ncbi.nlm.nih.gov/pubmed/16186247 "Safar JG, DeArmond SJ, Kociuba K, Deering C, Didorenko S, Bouzamondo-BernsteinE, Prusiner SB, Tremblay P. Prion clearance in bigenic mice. J Gen Virol. 2005Oct;86(Pt 10):2913-23. PubMed PMID: 16186247."
[Samocha 2014]: http://www.ncbi.nlm.nih.gov/pubmed/25086666 "Samocha KE, Robinson EB, Sanders SJ, Stevens C, Sabo A, McGrath LM, KosmickiJA, Rehnström K, Mallick S, Kirby A, Wall DP, MacArthur DG, Gabriel SB, DePristo M, Purcell SM, Palotie A, Boerwinkle E, Buxbaum JD, Cook EH Jr, Gibbs RA,Schellenberg GD, Sutcliffe JS, Devlin B, Roeder K, Neale BM, Daly MJ. A frameworkfor the interpretation of de novo mutation in human disease. Nat Genet. 2014Sep;46(9):944-50. doi: 10.1038/ng.3050. Epub 2014 Aug 3. PubMed PMID: 25086666;PubMed Central PMCID: PMC4222185."
[Servick 2015]: http://www.ncbi.nlm.nih.gov/pubmed/26404822 "Servick K. Can 23andMe have it all? Science. 2015 Sep 25;349(6255):1472-4,1476-7. doi: 10.1126/science.349.6255.1472. PubMed PMID: 26404822."
[Shi 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23764840 "Shi Q, Chen C, Wang XJ, Zhou W, Wang JC, Zhang BY, Gao C, Gao C, Han J, DongXP. Rare V203I mutation in the PRNP gene of a Chinese patient withCreutzfeldt-Jakob disease. Prion. 2013 May-Jun;7(3):259-62. doi:10.4161/pri.24674. PubMed PMID: 23764840; PubMed Central PMCID: PMC3783113."
[Shi 2015]: http://www.ncbi.nlm.nih.gov/pubmed/26488179 "Shi Q, Zhou W, Chen C, Zhang BY, Xiao K, Zhang XC, Shen XJ, Li Q, Deng LQ,Dong JH, Lin WQ, Huang P, Jiang WJ, Lv J, Han J, Dong XP. The Features of GeneticPrion Diseases Based on Chinese Surveillance Program. PLoS One. 2015 Oct21;10(10):e0139552. doi: 10.1371/journal.pone.0139552. eCollection 2015. PubMedPMID: 26488179; PubMed Central PMCID: PMC4619501."
[Shibuya 1998]: http://www.ncbi.nlm.nih.gov/pubmed/9629853 "Shibuya S, Higuchi J, Shin RW, Tateishi J, Kitamoto T. Codon 219 Lys allele ofPRNP is not found in sporadic Creutzfeldt-Jakob disease. Ann Neurol. 1998Jun;43(6):826-8. PubMed PMID: 9629853."
[Silber 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24530226 "Silber BM, Gever JR, Rao S, Li Z, Renslo AR, Widjaja K, Wong C, Giles K,Freyman Y, Elepano M, Irwin JJ, Jacobson MP, Prusiner SB. Novel compoundslowering the cellular isoform of the human prion protein in cultured human cells.Bioorg Med Chem. 2014 Mar 15;22(6):1960-72. doi: 10.1016/j.bmc.2014.01.001. Epub 2014 Jan 9. PubMed PMID: 24530226; PubMed Central PMCID: PMC3984052."
[Simon 2000]: http://www.ncbi.nlm.nih.gov/pubmed/10665501 "Simon ES, Kahana E, Chapman J, Treves TA, Gabizon R, Rosenmann H, Zilber N,Korczyn AD. Creutzfeldt-Jakob disease profile in patients homozygous for the PRNPE200K mutation. Ann Neurol. 2000 Feb;47(2):257-60. PubMed PMID: 10665501."
[Simpson 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23857164 "Simpson M, Johanssen V, Boyd A, Klug G, Masters CL, Li QX, Pamphlett R, McLeanC, Lewis V, Collins SJ. Unusual clinical and molecular-pathological profile ofgerstmann-Sträussler-Scheinker disease associated with a novel PRNP mutation(V176G). JAMA Neurol. 2013 Sep 1;70(9):1180-5. doi: 10.1001/jamaneurol.2013.165. PubMed PMID: 23857164."
[Spudich 1995]: http://www.ncbi.nlm.nih.gov/pubmed/8529127 "Spudich S, Mastrianni JA, Wrensch M, Gabizon R, Meiner Z, Kahana I, Rosenmann H, Kahana E, Prusiner SB. Complete penetrance of Creutzfeldt-Jakob disease inLibyan Jews carrying the E200K mutation in the prion protein gene. Mol Med. 1995 Sep;1(6):607-13. PubMed PMID: 8529127; PubMed Central PMCID: PMC2229975."
[Stenson 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24077912 "Stenson PD, Mort M, Ball EV, Shaw K, Phillips A, Cooper DN. The Human GeneMutation Database: building a comprehensive mutation repository for clinical and molecular genetics, diagnostic testing and personalized genomic medicine. HumGenet. 2014 Jan;133(1):1-9. Review. PubMed PMID: 24077912; PubMed Central PMCID: PMC3898141."
[Stohr 2011]: http://www.ncbi.nlm.nih.gov/pubmed/22160704 "Stöhr J, Watts JC, Legname G, Oehler A, Lemus A, Nguyen HO, Sussman J, WilleH, DeArmond SJ, Prusiner SB, Giles K. Spontaneous generation of anchorless prionsin transgenic mice. Proc Natl Acad Sci U S A. 2011 Dec 27;108(52):21223-8. doi:10.1073/pnas.1117827108. Epub 2011 Dec 12. PubMed PMID: 22160704; PubMed Central PMCID: PMC3248514."
[Tartaglia 2010]: http://www.ncbi.nlm.nih.gov/pubmed/21107135 "Tartaglia MC, Thai JN, See T, Kuo A, Harbaugh R, Raudabaugh B, Cali I,Sattavat M, Sanchez H, DeArmond SJ, Geschwind MD. Pathologic evidence that theT188R mutation in PRNP is associated with prion disease. J Neuropathol ExpNeurol. 2010 Dec;69(12):1220-7. doi: 10.1097/NEN.0b013e3181ffc39c. PubMed PMID:21107135; PubMed Central PMCID: PMC3136530."
[Tateishi 1990]: http://www.ncbi.nlm.nih.gov/pubmed/1699173 "Tateishi J, Kitamoto T, Doh-ura K, Sakaki Y, Steinmetz G, Tranchant C, Warter JM, Heldt N. Immunochemical, molecular genetic, and transmission studies on acase of Gerstmann-Sträussler-Scheinker syndrome. Neurology. 1990Oct;40(10):1578-81. PubMed PMID: 1699173."
[Tunnell 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18955686 "Tunnell E, Wollman R, Mallik S, Cortes CJ, Dearmond SJ, Mastrianni JA. A novelPRNP-P105S mutation associated with atypical prion disease and a rare PrPScconformation. Neurology. 2008 Oct 28;71(18):1431-8. doi:10.1212/01.wnl.0000330237.94742.fa. PubMed PMID: 18955686; PubMed Central PMCID: PMC2676963."
[Tunnell 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18955686 "Tunnell E, Wollman R, Mallik S, Cortes CJ, Dearmond SJ, Mastrianni JA. A novelPRNP-P105S mutation associated with atypical prion disease and a rare PrPScconformation. Neurology. 2008 Oct 28;71(18):1431-8. doi:10.1212/01.wnl.0000330237.94742.fa. PubMed PMID: 18955686; PubMed Central PMCID: PMC2676963."
[Vita 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23979103 "Vita MG, Gaudino S, Di Giuda D, Sauchelli D, Alboini PE, Gangemi E, BizzarroA, Scaricamazza E, Capellari S, Parchi P, Masullo C. R208H-129VV haplotype in theprion protein gene: phenotype and neuroimaging of a patient with geneticCreutzfeldt-Jakob disease. J Neurol. 2013 Oct;260(10):2650-2. doi:10.1007/s00415-013-7078-9. Epub 2013 Aug 25. Erratum in: J 2013 Dec;260(12):3199.PubMed PMID: 23979103."
[Webb 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18757886 "Webb TE, Poulter M, Beck J, Uphill J, Adamson G, Campbell T, Linehan J, PowellC, Brandner S, Pal S, Siddique D, Wadsworth JD, Joiner S, Alner K, Petersen C,Hampson S, Rhymes C, Treacy C, Storey E, Geschwind MD, Nemeth AH, Wroe S,Collinge J, Mead S. Phenotypic heterogeneity and genetic modification of P102Linherited prion disease in an international series. Brain. 2008 Oct;131(Pt10):2632-46. doi: 10.1093/brain/awn202. Epub 2008 Aug 30. PubMed PMID: 18757886; PubMed Central PMCID: PMC2570713."
[Webb 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18757886 "Webb TE, Poulter M, Beck J, Uphill J, Adamson G, Campbell T, Linehan J, PowellC, Brandner S, Pal S, Siddique D, Wadsworth JD, Joiner S, Alner K, Petersen C,Hampson S, Rhymes C, Treacy C, Storey E, Geschwind MD, Nemeth AH, Wroe S,Collinge J, Mead S. Phenotypic heterogeneity and genetic modification of P102Linherited prion disease in an international series. Brain. 2008 Oct;131(Pt10):2632-46. doi: 10.1093/brain/awn202. Epub 2008 Aug 30. PubMed PMID: 18757886; PubMed Central PMCID: PMC2570713."
[White 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18632556 "White MD, Farmer M, Mirabile I, Brandner S, Collinge J, Mallucci GR. Singletreatment with RNAi against prion protein rescues early neuronal dysfunction and prolongs survival in mice with prion disease. Proc Natl Acad Sci U S A. 2008 Jul 22;105(29):10238-43. doi: 10.1073/pnas.0802759105. Epub 2008 Jul 16. PubMed PMID:18632556; PubMed Central PMCID: PMC2474561."
[Windl 1999]: http://www.ncbi.nlm.nih.gov/pubmed/10987652 "Windl O, Giese A, Schulz-Schaeffer W, Zerr I, Skworc K, Arendt S, Oberdieck C,Bodemer M, Poser S, Kretzschmar HA. Molecular genetics of human prion diseases inGermany. Hum Genet. 1999 Sep;105(3):244-52. PubMed PMID: 10987652."
[Woolf 1955]: http://www.ncbi.nlm.nih.gov/pubmed/14388528 "WOOLF B. On estimating the relation between blood group and disease. Ann HumGenet. 1955 Jun;19(4):251-3. PubMed PMID: 14388528."
[Yamada 1993]: http://www.ncbi.nlm.nih.gov/pubmed/7902972 "Yamada M, Itoh Y, Fujigasaki H, Naruse S, Kaneko K, Kitamoto T, Tateishi J,Otomo E, Hayakawa M, Tanaka J, et al. A missense mutation at codon 105 with codon129 polymorphism of the prion protein gene in a new variant ofGerstmann-Sträussler-Scheinker disease. Neurology. 1993 Dec;43(12):2723-4. PubMedPMID: 7902972."
[Yamada 1999]: http://www.ncbi.nlm.nih.gov/pubmed/10408557 "Yamada M, Itoh Y, Inaba A, Wada Y, Takashima M, Satoh S, Kamata T, Okeda R,Kayano T, Suematsu N, Kitamoto T, Otomo E, Matsushita M, Mizusawa H. An inheritedprion disease with a PrP P105L mutation: clinicopathologic and PrP heterogeneity.Neurology. 1999 Jul 13;53(1):181-8. Review. PubMed PMID: 10408557."
[Yang 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19675240 "Yang W, Cook J, Rassbach B, Lemus A, DeArmond SJ, Mastrianni JA. A NewTransgenic Mouse Model of Gerstmann-Straussler-Scheinker Syndrome Caused by theA117V Mutation of PRNP. J Neurosci. 2009 Aug 12;29(32):10072-80. doi:10.1523/JNEUROSCI.2542-09.2009. PubMed PMID: 19675240; PubMed Central PMCID:PMC2749997."
[Yu 2009]: http://www.ncbi.nlm.nih.gov/pubmed/18951376 "Yu G, Chen J, Xu Y, Zhu C, Yu H, Liu S, Sha H, Chen J, Xu X, Wu Y, Zhang A, MaJ, Cheng G. Generation of goats lacking prion protein. Mol Reprod Dev. 2009Jan;76(1):3. doi: 10.1002/mrd.20960. PubMed PMID: 18951376."
[Zerr 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19773352 "Zerr I, Kallenberg K, Summers DM, Romero C, Taratuto A, Heinemann U,Breithaupt M, Varges D, Meissner B, Ladogana A, Schuur M, Haik S, Collins SJ,Jansen GH, Stokin GB, Pimentel J, Hewer E, Collie D, Smith P, Roberts H, Brandel JP, van Duijn C, Pocchiari M, Begue C, Cras P, Will RG, Sanchez-Juan P. Updatedclinical diagnostic criteria for sporadic Creutzfeldt-Jakob disease. Brain. 2009 Oct;132(Pt 10):2659-68. doi: 10.1093/brain/awp191. Epub 2009 Sep 22. Erratum in: Brain. 2012 Apr;135(Pt 4):1335. PubMed PMID: 19773352; PubMed Central PMCID:PMC2759336."
[Zhang 2014]: http://www.ncbi.nlm.nih.gov/pubmed/23787189 "Zhang H, Wang M, Wu L, Zhang H, Jin T, Wu J, Sun L. Novel prion protein genemutation at codon 196 (E196A) in a septuagenarian with Creutzfeldt-Jakob disease.J Clin Neurosci. 2014 Jan;21(1):175-8. doi: 10.1016/j.jocn.2013.03.016. Epub 2013Jun 17. PubMed PMID: 23787189."
[Zheng 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18425766 "Zheng L, Longfei J, Jing Y, Xinqing Z, Haiqing S, Haiyan L, Fen W, Xiumin D,Jianping J. PRNP mutations in a series of apparently sporadic neurodegenerativedementias in China. Am J Med Genet B Neuropsychiatr Genet. 2008 Sep5;147B(6):938-44. doi: 10.1002/ajmg.b.30761. PubMed PMID: 18425766."



[Durand 2014b]: http://biorxiv.org/content/early/2014/10/18/010512
[Lek 2015]: http://biorxiv.org/content/early/2015/10/30/030338 "Exome Aggregation Consortium, Monkol Lek, Konrad Karczewski, Eric Minikel, Kaitlin Samocha, Eric Banks, Timothy Fennell, Anne O'Donnell-Luria, James Ware, Andrew Hill, Beryl Cummings, Taru Tukiainen, Daniel Birnbaum, Jack Kosmicki, Laramie Duncan, Karol Estrada, Fengmei Zhao, James Zou, Emma Pierce-Hoffman, David Cooper, Mark DePristo, Ron Do, Jason Flannick, Menachem Fromer, Laura Gauthier, Jackie Goldstein, Namrata Gupta, Daniel Howrigan, Adam Kiezun, Mitja Kurki, Ami Levy Moonshine, Pradeep Natarajan, Lorena Orozco, Gina Peloso, Ryan Poplin, Manuel Rivas, Valentin Ruano-Rubio, Douglas Ruderfer, Khalid Shakir, Peter Stenson, Christine Stevens, Brett Thomas, Grace Tiao, Maria Tusie-Luna, Ben Weisburd, Hong-Hee Won, Dongmei Yu, David Altshuler, Diego Ardissino, Michael Boehnke, John Danesh, Elosua Roberto, Jose Florez, Stacey Gabriel, Gad Getz, Christina Hultman, Sekar Kathiresan, Markku Laakso, Steven McCarroll, Mark McCarthy, Dermot McGovern, Ruth McPherson, Benjamin Neale, Aarno Palotie, Shaun Purcell, Danish Saleheen, Jeremiah Scharf, Pamela Sklar, Sullivan Patrick, Jaakko Tuomilehto, Hugh Watkins, James Wilson, Mark Daly, Daniel MacArthur. Analysis of protein-coding genetic variation in 60,706 humans. bioRxiv. doi: http://dx.doi.org/10.1101/030338"
[EUROCJD Surveillance Data]: http://web.archive.org/web/20151102144718/http://www.eurocjd.ed.ac.uk/surveillance%20data%201.html "EUROCJD Surveillance Data, accessed November 2, 2015."
[UN Population and Vital Statistics Report]: http://unstats.un.org/unsd/demographic/products/vitstats/ "United Nations Statistics Divison. Population and Vital Statistics Report."
[CJD US 2015]: http://web.archive.org/web/20150401163801/http://www.cjdsurveillance.com/pdf/web_table.pdf
[CJD UK 2015]: http://web.archive.org/web/20150330211505/http://www.cjd.ed.ac.uk/documents/figs.pdf
[Kong 2004]: https://cshmonographs.org/index.php/monographs/article/view/4035
[Nozaki 2011]: http://brain.oxfordjournals.org/content/brain/early/2011/11/16/brain.awr295.full.pdf
[Li 1961]: http://catalog.hathitrust.org/Record/001496005 "C.-C. Li, Human genetics: principles and methods. (The Blakiston Division, McGraw-Hill Book Company, Inc., New York, 1961; http://catalog.hathitrust.org/Record/001496005)"


